<!DOCTYPE html>
<html>
<head>
  <meta charset="UTF-8">
  <link href="shared/bookhub.css" rel="stylesheet" type="text/css">
  <title>Amino Acids, Proteins, and Enzymes</title>
</head>
<body>

  
  <div id=navbar-top class="navbar">
    <div class="navbar-part left">
      
        <a href="s20-lipids.html"><img src="shared/images/batch-left.png"></a> <a href="s20-lipids.html">Previous Chapter</a>
      
    </div>
    <div class="navbar-part middle">
      <a href="index.html"><img src="shared/images/batch-up.png"></a> <a href="index.html">Table of Contents</a>
    </div>
    <div class="navbar-part right">
      
        <a href="s22-nucleic-acids.html">Next Chapter</a> <a href="s22-nucleic-acids.html"><img src="shared/images/batch-right.png"></a>
      
    </div>
  </div>

  <div id="book-content">
    <div class="chapter" id="gob-ch18" version="5.0" lang="en">
    <h1 class="title editable block">
<span class="title-prefix">Chapter 18</span> Amino Acids, Proteins, and Enzymes</h1>
    <div class="callout block" id="gob-ch18_n01">
        <h3 class="title">Opening Essay</h3>
        <p class="para" id="gob-ch18_p01">The 1923 Nobel Prize in Medicine or Physiology was awarded to Frederick Grant Banting and John James Richard Macleod for their discovery of the protein <em class="emphasis">insulin</em>. In 1958, the Nobel Prize in Chemistry was awarded to Frederick Sanger for his discoveries concerning the structure of proteins and, in particular, the structure of insulin. What is so important about insulin that two Nobel Prizes have been awarded for work on this protein?</p>
        <p class="para" id="gob-ch18_p02">Insulin is a hormone that is synthesized in the pancreas. (For more information about hormones, see <a class="xref" href="gob-ch17#gob-ch17">Chapter 17 "Lipids"</a>, <a class="xref" href="gob-ch17_s04#gob-ch17_s04">Section 17.4 "Steroids"</a>.) Insulin stimulates the transport of glucose into cells throughout the body and the storage of glucose as glycogen. People with diabetes do not produce insulin or use it properly. The isolation of insulin in 1921 led to the first effective treatment for these individuals.</p>
        <div class="figure large medium-height" id="gob-ch18_f01">
            <p class="title"><span class="title-prefix">Figure 18.1</span> An Insulin Pump</p>
            <img src="section_21/e7bf9545111e5a043b8df7b6590373d6.jpg">
        </div>
    </div>
    <p class="para editable block" id="gob-ch18_p03"><span class="margin_term"><a class="glossterm">Proteins</a><span class="glossdef">A compound of high molar mass consisting largely or entirely of amino acids linked together.</span></span> may be defined as compounds of high molar mass consisting largely or entirely of chains of amino acids. Their masses range from several thousand to several million daltons (Da). In addition to carbon, hydrogen, and oxygen atoms, all proteins contain nitrogen and sulfur atoms, and many also contain phosphorus atoms and traces of other elements. Proteins serve a variety of roles in living organisms and are often classified by these biological roles, which are summarized in <a class="xref" href="#gob-ch18_t01">Table 18.1 "Classification of Proteins by Biological Function"</a>. Muscle tissue is largely protein, as are skin and hair. Proteins are present in the blood, in the brain, and even in tooth enamel. Each type of cell in our bodies makes its own specialized proteins, as well as proteins common to all or most cells.</p>
    <div class="callout editable block" id="gob-ch18_n02">
        <h3 class="title">Note</h3>
        <p class="para" id="gob-ch18_p04">The <em class="emphasis">dalton</em> is a unit of mass used by biochemists and biologists. It is equivalent to the atomic mass unit. A 30,000 Da protein has a molar mass of 30,000 u.</p>
    </div>
    <div class="table block" id="gob-ch18_t01" frame="all">
        <p class="title"><span class="title-prefix">Table 18.1</span> Classification of Proteins by Biological Function</p>
        <table cellpadding="0" cellspacing="0">
            <thead>
                <tr>
                    <th>Classification</th>
                    <th>Biological Function</th>
                    <th>Example</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>enzymes</td>
                    <td>accelerate biological reactions</td>
                    <td>α-Amylase catalyzes the hydrolysis of starch and glycogen.</td>
                </tr>
                <tr>
                    <td>structural</td>
                    <td>provide strength and structure</td>
                    <td>Keratin is the primary protein of hair and wool.</td>
                </tr>
                <tr>
                    <td>contractile</td>
                    <td>muscle contraction; cell division</td>
                    <td>Myosin is one protein needed for the contraction of muscles.</td>
                </tr>
                <tr>
                    <td>transport</td>
                    <td>transport substances from one place to another</td>
                    <td>Hemoglobin transports oxygen from the lungs throughout the body.</td>
                </tr>
                <tr>
                    <td>regulatory</td>
                    <td>regulate the functioning of other proteins</td>
                    <td>Insulin regulates the activity of specific enzymes in the body.</td>
                </tr>
                <tr>
                    <td>storage</td>
                    <td>provide storage of essential nutrients</td>
                    <td>Ovalbumin stores amino acids in the egg white that will be used by the developing bird.</td>
                </tr>
                <tr>
                    <td>protection</td>
                    <td>protect cells or the organism from foreign substances</td>
                    <td>Immunoglobulins recognize and breakdown foreign molecules.</td>
                </tr>
            </tbody>
        </table>
    </div>
    <p class="para editable block" id="gob-ch18_p05">We begin our study of proteins by looking at the properties and reactions of amino acids, which is followed by a discussion of how amino acids link covalently to form peptides and proteins. We end the chapter with a discussion of enzymes—the proteins that act as catalysts in the body.</p>
</div>
<div class="section" id="gob-ch18_s01" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.1</span> Properties of Amino Acids</h2>
    <div class="learning_objectives editable block" id="gob-ch18_s01_n01">
        <h3 class="title">Learning Objective</h3>
        <ol class="orderedlist" id="gob-ch18_s01_l01">
            <li>Recognize amino acids and classify them based on the characteristics of their side chains.</li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s01_p01">The proteins in all living species, from bacteria to humans, are constructed from the same set of 20 <span class="margin_term"><a class="glossterm">amino acids</a><span class="glossdef">A molecule that contains an amino group and a carboxyl group.</span></span>, so called because each contains an amino group attached to a carboxylic acid. (For more information about amino groups, see <a class="xref" href="gob-ch15#gob-ch15">Chapter 15 "Organic Acids and Bases and Some of Their Derivatives"</a>, <a class="xref" href="gob-ch15_s01#gob-ch15_s01">Section 15.1 "Functional Groups of the Carboxylic Acids and Their Derivatives"</a>.) The amino acids in proteins are α-amino acids, which means the amino group is attached to the α-carbon of the carboxylic acid. (For more information about the α-carbon, see <a class="xref" href="gob-ch15#gob-ch15">Chapter 15 "Organic Acids and Bases and Some of Their Derivatives"</a>, <a class="xref" href="gob-ch15_s02#gob-ch15_s02">Section 15.2 "Carboxylic Acids: Structures and Names"</a>.) Humans can synthesize only about half of the needed amino acids; the remainder must be obtained from the diet and are known as <span class="margin_term"><a class="glossterm">essential amino acids</a><span class="glossdef">An amino acid that must be obtained from the diet because it cannot be synthesized in sufficient quantities by the body.</span></span>.</p>
    <div class="callout editable block" id="gob-ch18_s01_n02">
        <h3 class="title">Note</h3>
        <p class="para" id="gob-ch18_s01_p02">Two more amino acids have been found in limited quantities in proteins. Selenocysteine was discovered in 1986, while pyrrolysine was discovered in 2002.</p>
    </div>
    <p class="para editable block" id="gob-ch18_s01_p03">The amino acids are colorless, nonvolatile, crystalline solids, melting and decomposing at temperatures above 200°C. These melting temperatures are more like those of inorganic salts than those of amines or organic acids and indicate that the structures of the amino acids in the solid state and in neutral solution are best represented as having both a negatively charged group and a positively charged group. Such a species is known as a <span class="margin_term"><a class="glossterm">zwitterion</a><span class="glossdef">An electrically neutral compound that contains both negatively and positively charged groups.</span></span>.</p>
    <div class="informalfigure large block">
        <img src="section_21/6ce67a97fa28a44e9983fa12939b196a.jpg">
    </div>
    <div class="section" id="gob-ch18_s01_s01">
        <h2 class="title editable block">Classification</h2>
        <p class="para editable block" id="gob-ch18_s01_s01_p01">In addition to the amino and carboxyl groups, amino acids have a side chain or R group attached to the α-carbon. Each amino acid has unique characteristics arising from the size, shape, solubility, and ionization properties of its R group. As a result, the side chains of amino acids exert a profound effect on the structure and biological activity of proteins. Although amino acids can be classified in various ways, one common approach is to classify them according to whether the functional group on the side chain at neutral pH is nonpolar, polar but uncharged, negatively charged, or positively charged. The structures and names of the 20 amino acids, their one- and three-letter abbreviations, and some of their distinctive features are given in <a class="xref" href="#gob-ch18_s01_s01_t01">Table 18.2 "Common Amino Acids Found in Proteins"</a>.</p>
        <div class="table block" id="gob-ch18_s01_s01_t01" frame="all">
            <p class="title"><span class="title-prefix">Table 18.2</span> Common Amino Acids Found in Proteins</p>
            <table cellpadding="0" cellspacing="0">
                <thead>
                    <tr>
                        <th>Common Name</th>
                        <th>Abbreviation</th>
                        <th>Structural Formula (at pH 6)</th>
                        <th>Molar Mass</th>
                        <th>Distinctive Feature</th>
                    </tr>
                </thead>
                <tbody>
                    <tr>
                        <td colspan="5"><strong class="emphasis bold">Amino acids with a nonpolar R group</strong></td>
                    </tr>
                    <tr>
                        <td>glycine</td>
                        <td>gly (G)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/7fe184bc1edcb8cf40851bb62cfb7182.jpg">
                </div>
</td>
                        <td>75</td>
                        <td>the only amino acid lacking a chiral carbon</td>
                    </tr>
                    <tr>
                        <td>alanine</td>
                        <td>ala (A)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/f0327ddd85f14253ec7c3e902077adcc.jpg">
                </div>
</td>
                        <td>89</td>
                        <td>—</td>
                    </tr>
                    <tr>
                        <td>valine</td>
                        <td>val (V)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/da1aa9e6507b67f16936a98ab3e062c1.jpg">
                </div>
</td>
                        <td>117</td>
                        <td>a branched-chain amino acid</td>
                    </tr>
                    <tr>
                        <td>leucine</td>
                        <td>leu (L)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/7feae33da978f4e4d15b8a47e9e8be46.jpg">
                </div>
</td>
                        <td>131</td>
                        <td>a branched-chain amino acid</td>
                    </tr>
                    <tr>
                        <td>isoleucine</td>
                        <td>ile (I)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/016c5c25430dd25bfdd64db5a18282ad.jpg">
                </div>
</td>
                        <td>131</td>
                        <td>an essential amino acid because most animals cannot synthesize branched-chain amino acids</td>
                    </tr>
                    <tr>
                        <td>phenylalanine</td>
                        <td>phe (F)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/ea72e274624f3ddfb3d2553f710714e3.jpg">
                </div>
</td>
                        <td>165</td>
                        <td>also classified as an aromatic amino acid</td>
                    </tr>
                    <tr>
                        <td>tryptophan</td>
                        <td>trp (W)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/0179041c9d7240954fedc2914a9acd6e.jpg">
                </div>
</td>
                        <td>204</td>
                        <td>also classified as an aromatic amino acid</td>
                    </tr>
                    <tr>
                        <td>methionine</td>
                        <td>met (M)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/0fa1e5f19c0783df3708095168b30a05.jpg">
                </div>
</td>
                        <td>149</td>
                        <td>side chain functions as a methyl group donor</td>
                    </tr>
                    <tr>
                        <td>proline</td>
                        <td>pro (P)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/cb08ea4ca52c16e52f1c1a95d9088f47.jpg">
                </div>
</td>
                        <td>115</td>
                        <td>contains a secondary amine group; referred to as an <em class="emphasis">α-imino acid</em>
</td>
                    </tr>
                    <tr>
                        <td colspan="5"><strong class="emphasis bold">Amino acids with a polar but neutral R group</strong></td>
                    </tr>
                    <tr>
                        <td>serine</td>
                        <td>ser (S)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/63848af5c32997eb781c2825ef7b146e.jpg">
                </div>
</td>
                        <td>105</td>
                        <td>found at the active site of many enzymes</td>
                    </tr>
                    <tr>
                        <td>threonine</td>
                        <td>thr (T)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/ce4f6352d122b852daabf06990887c35.jpg">
                </div>
</td>
                        <td>119</td>
                        <td>named for its similarity to the sugar threose</td>
                    </tr>
                    <tr>
                        <td>cysteine</td>
                        <td>cys (C)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/e50d80c23bf23cf172825879cce34c4b.jpg">
                </div>
</td>
                        <td>121</td>
                        <td>oxidation of two cysteine molecules yields <em class="emphasis">cystine</em>
</td>
                    </tr>
                    <tr>
                        <td>tyrosine</td>
                        <td>tyr (Y)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/7209f7a3ae9adfa439347cdcb8ea7fd5.jpg">
                </div>
</td>
                        <td>181</td>
                        <td>also classified as an aromatic amino acid</td>
                    </tr>
                    <tr>
                        <td>asparagine</td>
                        <td>asn (N)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/2eeebc4421b578fd768f3c89ed753971.jpg">
                </div>
</td>
                        <td>132</td>
                        <td>the amide of aspartic acid</td>
                    </tr>
                    <tr>
                        <td>glutamine</td>
                        <td>gln (Q)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/c786e6a0db62b979a69a4b9e768cdce1.jpg">
                </div>
</td>
                        <td>146</td>
                        <td>the amide of glutamic acid</td>
                    </tr>
                    <tr>
                        <td colspan="5"><strong class="emphasis bold">Amino acids with a negatively charged R group</strong></td>
                    </tr>
                    <tr>
                        <td>aspartic acid</td>
                        <td>asp (D)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/daea79b248b5e0710ef85691aae2d2a9.jpg">
                </div>
</td>
                        <td>132</td>
                        <td>carboxyl groups are ionized at physiological pH; also known as aspartate</td>
                    </tr>
                    <tr>
                        <td>glutamic acid</td>
                        <td>glu (E)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/d5f6aea2aff13631e7ab0de2e293833b.jpg">
                </div>
</td>
                        <td>146</td>
                        <td>carboxyl groups are ionized at physiological pH; also known as glutamate</td>
                    </tr>
                    <tr>
                        <td colspan="5"><strong class="emphasis bold">Amino acids with a positively charged R group</strong></td>
                    </tr>
                    <tr>
                        <td>histidine</td>
                        <td>his (H)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/f18c38bbc19d1bc05232e74d0e7e9f25.jpg">
                </div>
</td>
                        <td>155</td>
                        <td>the only amino acid whose R group has a pK<sub class="subscript">a</sub> (6.0) near physiological pH</td>
                    </tr>
                    <tr>
                        <td>lysine</td>
                        <td>lys (K)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/965e267b9d5ba8e56a6e913503113bf4.jpg">
                </div>
</td>
                        <td>147</td>
                        <td>—</td>
                    </tr>
                    <tr>
                        <td>arginine</td>
                        <td>arg (R)</td>
                        <td>
		                <div class="informalfigure small">
                    <img src="section_21/4857f12bc44f0966b40c0abdb4bd7f5e.jpg">
                </div>
</td>
                        <td>175</td>
                        <td>almost as strong a base as sodium hydroxide</td>
                    </tr>
                </tbody>
            </table>
        </div>
        <p class="para editable block" id="gob-ch18_s01_s01_p02">The first amino acid to be isolated was asparagine in 1806. It was obtained from protein found in asparagus juice (hence the name). Glycine, the major amino acid found in gelatin, was named for its sweet taste (Greek <em class="emphasis">glykys</em>, meaning “sweet”). In some cases an amino acid found in a protein is actually a derivative of one of the common 20 amino acids (one such derivative is hydroxyproline). The modification occurs <em class="emphasis">after</em> the amino acid has been assembled into a protein.</p>
        <div class="informalfigure large block">
            <img src="section_21/a3f6da5e3fea8fb194d5fdfe7696e43f.jpg">
        </div>
    </div>
    <div class="section" id="gob-ch18_s01_s02">
        <h2 class="title editable block">Configuration</h2>
        <p class="para editable block" id="gob-ch18_s01_s02_p01">Notice in <a class="xref" href="#gob-ch18_s01_s01_t01">Table 18.2 "Common Amino Acids Found in Proteins"</a> that glycine is the only amino acid whose α-carbon is <em class="emphasis">not</em> chiral. Therefore, with the exception of glycine, the amino acids could theoretically exist in either the D- or the L-enantiomeric form and rotate plane-polarized light. As with sugars, chemists use glyceraldehyde as the reference compound for the assignment of configuration to amino acids. (For more information about stereoisomers and configuration, see <a class="xref" href="gob-ch16#gob-ch16">Chapter 16 "Carbohydrates"</a>, <a class="xref" href="gob-ch16_s02#gob-ch16_s02">Section 16.2 "Classes of Monosaccharides"</a>.) Its structure closely resembles an amino acid structure except that in the latter, an amino group takes the place of the OH group on the chiral carbon of the sugar.</p>
        <div class="informalfigure large block">
            <img src="section_21/b306017d8c8aad1f7d8748992c51dc76.jpg">
        </div>
        <p class="para editable block" id="gob-ch18_s01_s02_p02">We learned in <a class="xref" href="gob-ch16#gob-ch16">Chapter 16 "Carbohydrates"</a> that all naturally occurring sugars belong to the D series. It is interesting, therefore, that nearly all known plant and animal proteins are composed entirely of L-amino acids. However, certain bacteria contain D-amino acids in their cell walls, and several antibiotics (e.g., actinomycin D and the gramicidins) contain varying amounts of D-leucine, D-phenylalanine, and D-valine.</p>
        <div class="qandaset block" id="gob-ch18_s01_qs01" condition="1col" defaultlabel="number">
            <h3 class="title">Concept Review Exercises</h3>
            <ol class="qandadiv" id="gob-ch18_s01_qs01_qd01">
                <li class="qandaentry" id="gob-ch18_s01_qs01_qd01_qa01">
                    <div class="question">
                        <p class="para" id="gob-ch18_s01_qs01_p01">What is the general structure of an α-amino acid?</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs01_qd01_qa02">
                    <div class="question">
                        <p class="para" id="gob-ch18_s01_qs01_p02">Identify the amino acid that fits each description.</p>
                        <ol class="orderedlist" id="gob-ch18_s01_qs01_l01" numeration="loweralpha">
                            <li>also known as aspartate</li>
                            <li>almost as strong a base as sodium hydroxide</li>
                            <li>does not have a chiral carbon</li>
                        </ol>
                    </div>
                </li>
            </ol>
        </div>
        <div class="qandaset block" id="gob-ch18_s01_qs01_ans" condition="1col" defaultlabel="number">
            <h3 class="title">Answers</h3>
            <ol class="qandadiv">
                <li class="qandaentry" id="gob-ch18_s01_qs01_qd01_qa01_ans">
                    <div class="answer">
                        <div class="informalfigure large">
                            <img src="section_21/32ec767b03ba868e7690790471981c37.jpg">
                        </div>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs01_qd01_qa02_ans">
                    <div class="answer">
                        <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s01_qs01_l02_ans" numeration="loweralpha">                                <li>aspartic acid</li>
                                <li>arginine</li>
                                <li>glycine</li>
</ol>
                    </div>
                </li>
            </ol>
        </div>
        <div class="key_takeaways editable block" id="gob-ch18_s01_s02_n02">
            <h3 class="title">Key Takeaways</h3>
            <ul class="itemizedlist" id="gob-ch18_s01_s02_l04">
                <li>Amino acids can be classified based on the characteristics of their distinctive side chains as nonpolar, polar but uncharged, negatively charged, or positively charged.</li>
                <li>The amino acids found in proteins are L-amino acids.</li>
            </ul>
        </div>
        <div class="qandaset block" id="gob-ch18_s01_qs02" defaultlabel="number">
            <h3 class="title">Exercises</h3>
            <ol class="qandadiv" id="gob-ch18_s01_qs02_qd01">
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa01">
                    <div class="question">
                        <p class="para" id="gob-ch18_s01_qs02_p01">Write the side chain of each amino acid.</p>
                        <ol class="orderedlist" id="gob-ch18_s01_qs02_l01" numeration="loweralpha">
                            <li>serine</li>
                            <li>arginine</li>
                            <li>phenylalanine</li>
                        </ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa02">
                    <div class="question">
                        <p class="para" id="gob-ch18_s01_qs02_p02">Write the side chain of each amino acid.</p>
                        <ol class="orderedlist" id="gob-ch18_s01_qs02_l03" numeration="loweralpha">
                            <li>aspartic acid</li>
                            <li>methionine</li>
                            <li>valine</li>
                        </ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa03">
                    <div class="question">
                        <p class="para" id="gob-ch18_s01_qs02_p03">Draw the structure for each amino acid.</p>
                        <ol class="orderedlist" id="gob-ch18_s01_qs02_l05" numeration="loweralpha">
                            <li>alanine</li>
                            <li>cysteine</li>
                            <li>histidine</li>
                        </ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa04">
                    <div class="question">
                        <p class="para" id="gob-ch18_s01_qs02_p04">Draw the structure for each amino acid.</p>
                        <ol class="orderedlist" id="gob-ch18_s01_qs02_l07" numeration="loweralpha">
                            <li>threonine</li>
                            <li>glutamic acid</li>
                            <li>leucine</li>
                        </ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa05">
                    <div class="question">
                        <p class="para" id="gob-ch18_s01_qs02_p05">Identify an amino acid whose side chain contains a(n)</p>
                        <ol class="orderedlist" id="gob-ch18_s01_qs02_l09" numeration="loweralpha">
                            <li>amide functional group.</li>
                            <li>aromatic ring.</li>
                            <li>carboxyl group.</li>
                        </ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa06">
                    <div class="question">
                        <p class="para" id="gob-ch18_s01_qs02_p06">Identify an amino acid whose side chain contains a(n)</p>
                        <ol class="orderedlist" id="gob-ch18_s01_qs02_l11" numeration="loweralpha">
                            <li>OH group</li>
                            <li>branched chain</li>
                            <li>amino group</li>
                        </ol>
                    </div>
                </li>
            </ol>
        </div>
        <div class="qandaset block" id="gob-ch18_s01_qs02_ans" defaultlabel="number">
            <h3 class="title">Answers</h3>
            <ol class="qandadiv">
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa01_ans">
                    <div class="answer">
                        <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s01_qs02_l02_ans" numeration="loweralpha">                                <li>CH<sub class="subscript">2</sub>OH<sup class="superscript">−</sup>
</li>
                                <li>
                                    <div class="informalfigure large">
                                        <img src="section_21/6eaea3a832a142af78264b67975dddd6.jpg">
                                    </div>
                                </li>
                                <li>
                                    <div class="informalfigure large">
                                        <img src="section_21/cdaad45a04a0232a53eedabdc67e97e5.jpg">
                                    </div>
                                </li>
</ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa02_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa03_ans">
                    <div class="answer">
                        <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s01_qs02_l06_ans" numeration="loweralpha">                                <li>
                                    <div class="informalfigure large">
                                        <img src="section_21/0b299efbd6cd00e04031007490bf5c0e.jpg">
                                    </div>
                                </li>
                                <li>
                                    <div class="informalfigure large">
                                        <img src="section_21/efec0b62a45280fb47d7890299d45c57.jpg">
                                    </div>
                                </li>
                                <li>
                                    <div class="informalfigure large">
                                        <img src="section_21/ad4985cc7c681c9b489dc0abea1c1e45.jpg">
                                    </div>
                                </li>
</ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa04_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa05_ans">
                    <div class="answer">
                        <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s01_qs02_l10_ans" numeration="loweralpha">                                <li>asparagine or glutamine</li>
                                <li>phenylalanine, tyrosine, or tryptophan</li>
                                <li>aspartic acid or glutamic acid</li>
</ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s01_qs02_qd01_qa06_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
            </ol>
        </div>
    </div>
</div>
<div class="section" id="gob-ch18_s02" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.2</span> Reactions of Amino Acids</h2>
    <div class="learning_objectives editable block" id="gob-ch18_s02_n01">
        <h3 class="title">Learning Objective</h3>
        <ol class="orderedlist" id="gob-ch18_s02_l01">
            <li>Explain how an amino acid can act as both an acid and a base.</li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s02_p01">The structure of an amino acid allows it to act as both an acid and a base. An amino acid has this ability because at a certain pH value (different for each amino acid) nearly all the amino acid molecules exist as zwitterions. If acid is added to a solution containing the zwitterion, the carboxylate group captures a hydrogen (H<sup class="superscript">+</sup>) ion, and the amino acid becomes positively charged. If base is added, ion removal of the H<sup class="superscript">+</sup> ion from the amino group of the zwitterion produces a negatively charged amino acid. In both circumstances, the amino acid acts to maintain the pH of the system—that is, to remove the added acid (H<sup class="superscript">+</sup>) or base (OH<sup class="superscript">−</sup>) from solution.</p>
    <div class="informalfigure large block">
        <img src="section_21/ec4991fcc176fcfc73905e203570504e.jpg">
    </div>
    <div class="exercises editable block" id="gob-ch18_s02_n02">
        <h3 class="title">Example 1</h3>
        <ol class="orderedlist" id="gob-ch18_s02_l02">
            <li>Draw the structure for the anion formed when glycine (at neutral pH) reacts with a base.</li>
            <li>Draw the structure for the cation formed when glycine (at neutral pH) reacts with an acid.</li>
        </ol>
        <p class="simpara">Solution</p>
        <ol class="orderedlist" id="gob-ch18_s02_l03">
            <li>
                <p class="para">The base removes H<sup class="superscript">+</sup> from the protonated amine group.</p>
                <div class="informalfigure large">
                    <img src="section_21/70b1c37e1481a9c9722d60b57a3400ec.jpg">
                </div>
            </li>
            <li>
                <p class="para">The acid adds H<sup class="superscript">+</sup> to the carboxylate group.</p>
                <div class="informalfigure large">
                    <img src="section_21/a951cb6209eeb790e728058752ab8b11.jpg">
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s02_qs01" condition="1col" defaultlabel="number">
        <h3 class="title">Skill-Building Exercise</h3>
        <ol class="qandadiv" id="gob-ch18_s02_qs01_qd01">
            <li class="qandaentry" id="gob-ch18_s02_qs01_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s02_qs01_p01">Draw the structure for the cation formed when valine (at neutral pH) reacts with an acid.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s02_qs01_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s02_qs01_p02">Draw the structure for the anion formed when valine (at neutral pH) reacts with a base.</p>
                </div>
            </li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s02_p04">The particular pH at which a given amino acid exists in solution as a zwitterion is called the <span class="margin_term"><a class="glossterm">isoelectric point</a><span class="glossdef">The pH at which a given amino acid exists in solution as a zwitterion.</span></span> (pI). At its pI, the positive and negative charges on the amino acid balance, and the molecule as a whole is electrically neutral. The amino acids whose side chains are always neutral have isoelectric points ranging from 5.0 to 6.5. The basic amino acids (which have positively charged side chains at neutral pH) have relatively high pIs. Acidic amino acids (which have negatively charged side chains at neutral pH) have quite low pIs (<a class="xref" href="#gob-ch18_s02_t01">Table 18.3 "pIs of Some Representative Amino Acids"</a>).</p>
    <div class="table block" id="gob-ch18_s02_t01" frame="all">
        <p class="title"><span class="title-prefix">Table 18.3</span> pIs of Some Representative Amino Acids</p>
        <table cellpadding="0" cellspacing="0">
            <thead>
                <tr>
                    <th>Amino Acid</th>
                    <th>Classification</th>
                    <th align="right">pI</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>alanine</td>
                    <td>nonpolar</td>
                    <td align="right">6.0</td>
                </tr>
                <tr>
                    <td>valine</td>
                    <td>nonpolar</td>
                    <td align="right">6.0</td>
                </tr>
                <tr>
                    <td>serine</td>
                    <td>polar, uncharged</td>
                    <td align="right">5.7</td>
                </tr>
                <tr>
                    <td>threonine</td>
                    <td>polar, uncharged</td>
                    <td align="right">6.5</td>
                </tr>
                <tr>
                    <td>arginine</td>
                    <td>positively charged (basic)</td>
                    <td align="right">10.8</td>
                </tr>
                <tr>
                    <td>histidine</td>
                    <td>positively charged (basic)</td>
                    <td align="right">7.6</td>
                </tr>
                <tr>
                    <td>lysine</td>
                    <td>positively charged (basic)</td>
                    <td align="right">9.8</td>
                </tr>
                <tr>
                    <td>aspartic acid</td>
                    <td>negatively charged (acidic)</td>
                    <td align="right">3.0</td>
                </tr>
                <tr>
                    <td>glutamic acid</td>
                    <td>negatively charged (acidic)</td>
                    <td align="right">3.2</td>
                </tr>
            </tbody>
        </table>
    </div>
    <p class="para editable block" id="gob-ch18_s02_p05">Amino acids undergo reactions characteristic of carboxylic acids and amines. The reactivity of these functional groups is particularly important in linking amino acids together to form peptides and proteins, as you will see later in this chapter. Simple chemical tests that are used to detect amino acids take advantage of the reactivity of these functional groups. An example is the ninhydrin test in which the amine functional group of α-amino acids reacts with ninhydrin to form purple-colored compounds. Ninhydrin is used to detect fingerprints because it reacts with amino acids from the proteins in skin cells transferred to the surface by the individual leaving the fingerprint.</p>
    <div class="informalfigure large block">
        <img src="section_21/2563dcc3db90b5aaef85fd6e82bf7e36.jpg">
    </div>
    <div class="qandaset block" id="gob-ch18_s02_qs02" condition="1col" defaultlabel="number">
        <h3 class="title">Concept Review Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s02_qs02_qd01">
            <li class="qandaentry" id="gob-ch18_s02_qs02_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s02_qs02_p01">Define each term.</p>
                    <ol class="orderedlist" id="gob-ch18_s02_qs02_l01" numeration="loweralpha">
                        <li>zwitterion</li>
                        <li>isoelectric point</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s02_qs02_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s02_qs02_p02">Draw the structure for the anion formed when alanine (at neutral pH) reacts with a base.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s02_qs02_qd01_qa03">
                <div class="question">
                    <p class="para" id="gob-ch18_s02_qs02_p03">Draw the structure for the cation formed when alanine (at neutral pH) reacts with an acid.</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s02_qs02_ans" condition="1col" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s02_qs02_qd01_qa01_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s02_qs02_l02_ans" numeration="loweralpha">                            <li>an electrically neutral compound that contains both negatively and positively charged groups</li>
                            <li>the pH at which a given amino acid exists in solution as a zwitterion</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s02_qs02_qd01_qa02_ans">
                <div class="answer">
                    <div class="informalfigure large">
                        <img src="section_21/d37c429cd9db2e7cd4bf8214eeb13e6e.jpg">
                    </div>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s02_qs02_qd01_qa03_ans">
                <div class="answer">
                    <div class="informalfigure large">
                        <img src="section_21/b4c84ccb1f8d3f5a2be724b557d59929.jpg">
                    </div>
                </div>
            </li>
        </ol>
    </div>
    <div class="key_takeaways editable block" id="gob-ch18_s02_n05">
        <h3 class="title">Key Takeaways</h3>
        <ul class="itemizedlist" id="gob-ch18_s02_l08">
            <li>Amino acids can act as both an acid and a base due to the presence of the amino and carboxyl functional groups.</li>
            <li>The pH at which a given amino acid exists in solution as a zwitterion is called the <em class="emphasis">isoelectric point</em> (pI).</li>
        </ul>
    </div>
    <div class="qandaset block" id="gob-ch18_s02_qs03" condition="1col" defaultlabel="number">
        <h3 class="title">Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s02_qs03_qd01">
            <li class="qandaentry" id="gob-ch18_s02_qs03_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s02_qs03_p01">Draw the structure of leucine and determine the charge on the molecule in a(n)</p>
                    <ol class="orderedlist" id="gob-ch18_s02_qs03_l01" numeration="loweralpha">
                        <li>acidic solution (pH = 1).</li>
                        <li>neutral solution (pH = 7).</li>
                        <li>a basic solution (pH = 11)</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s02_qs03_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s02_qs03_p02">Draw the structure of isoleucine and determine the charge on the molecule in a(n)</p>
                    <ol class="orderedlist" id="gob-ch18_s02_qs03_l03" numeration="loweralpha">
                        <li>acidic solution (pH = 1).</li>
                        <li>neutral solution (pH = 7).</li>
                        <li>basic solution (pH = 11).</li>
                    </ol>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s02_qs03_ans" condition="1col" defaultlabel="number">
        <h3 class="title">Answer</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s02_qs03_qd01_qa01_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s02_qs03_l02_ans" numeration="loweralpha">                            <li>
                                <div class="informalfigure large" condition="equation-image">
                                    <img src="section_21/c82b071735370e5853e45ead953ada18.jpg">
                                </div>
                            </li>
                            <li>
                                <div class="informalfigure large" condition="equation-image">
                                    <img src="section_21/f79eaa3f548da3d7481c6cbb6cd0d3b0.jpg">
                                </div>
                            </li>
                            <li>
                                <div class="informalfigure large" condition="equation-image">
                                    <img src="section_21/a4a388ef6c68d37174394d4c0d32aefd.jpg">
                                </div>
                            </li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s02_qs03_qd01_qa02_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
        </ol>
    </div>
</div>
<div class="section" id="gob-ch18_s03" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.3</span> Peptides</h2>
    <div class="learning_objectives editable block" id="gob-ch18_s03_n01">
        <h3 class="title">Learning Objectives</h3>
        <ol class="orderedlist" id="gob-ch18_s03_l01">
            <li>Explain how a peptide is formed from individual amino acids.</li>
            <li>Explain why the sequence of amino acids in a protein is important.</li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s03_p01">Two or more amino acids can join together into chains called peptides. In <a class="xref" href="gob-ch15#gob-ch15">Chapter 15 "Organic Acids and Bases and Some of Their Derivatives"</a>, <a class="xref" href="gob-ch15_s16#gob-ch15_s16">Section 15.16 "Formation of Amides"</a>, we discussed the reaction between ammonia and a carboxylic acid to form an amide. In a similar reaction, the amino group on one amino acid molecule reacts with the carboxyl group on another, releasing a molecule of water and forming an amide linkage:</p>
    <div class="informalfigure large block">
        <img src="section_21/dc2b9c0a2720a543097192d3a1584b40.jpg">
    </div>
    <p class="para editable block" id="gob-ch18_s03_p02">An amide bond joining two amino acid units is called a <span class="margin_term"><a class="glossterm">peptide bond</a><span class="glossdef">The amide bond joining two amino acid units in a peptide or protein.</span></span>. Note that the product molecule still has a reactive amino group on the left and a reactive carboxyl group on the right. These can react with additional amino acids to lengthen the peptide. The process can continue until thousands of units have joined, resulting in large proteins.</p>
    <div class="informalfigure large block">
        <img src="section_21/845e1892447f21d1b0abbece7930f344.jpg">
    </div>
    <p class="para editable block" id="gob-ch18_s03_p03">A chain consisting of only two amino acid units is called a <em class="emphasis">dipeptide</em>; a chain consisting of three is a <em class="emphasis">tripeptide</em>. By convention, peptide and protein structures are depicted with the amino acid whose amino group is free (the N-terminal end) on the left and the amino acid with a free carboxyl group (the C-terminal end) to the right.</p>
    <div class="informalfigure large block">
        <img src="section_21/e8fee7b1e9c1c1cff3447816e529e10f.jpg">
    </div>
    <p class="para editable block" id="gob-ch18_s03_p04">The general term <em class="emphasis">peptide</em> refers to an amino acid chain of unspecified length. However, chains of about 50 amino acids or more are usually called proteins or <span class="margin_term"><a class="glossterm">polypeptides</a><span class="glossdef">A chain of about 50 or more amino acids.</span></span>. In its physiologically active form, a protein may be composed of one or more polypeptide chains.</p>
    <p class="para editable block" id="gob-ch18_s03_p05">For peptides and proteins to be physiologically active, it is not enough that they incorporate certain amounts of specific amino acids. The order, or <em class="emphasis">sequence</em>, in which the amino acids are connected is also of critical importance. Bradykinin is a nine-amino acid peptide produced in the blood that has the following amino acid sequence:</p>
    <span class="informalequation block"><span class="mathphrase">arg-pro-pro-gly-phe-ser-pro-phe-arg</span></span>
    <p class="para editable block" id="gob-ch18_s03_p06">This peptide lowers blood pressure, stimulates smooth muscle tissue, increases capillary permeability, and causes pain. When the order of amino acids in bradykinin is reversed,</p>
    <span class="informalequation block"><span class="mathphrase">arg-phe-pro-ser-phe-gly-pro-pro-arg</span></span>
    <p class="para editable block" id="gob-ch18_s03_p07">the peptide resulting from this synthesis shows none of the activity of bradykinin.</p>
    <p class="para editable block" id="gob-ch18_s03_p08">Just as millions of different words are spelled with our 26-letter English alphabet, millions of different proteins are made with the 20 common amino acids. However, just as the English alphabet can be used to write gibberish, amino acids can be put together in the <em class="emphasis">wrong sequence</em> to produce nonfunctional proteins. Although the correct sequence is ordinarily of utmost importance, it is not always absolutely required. Just as you can sometimes make sense of incorrectly spelled English words, a protein with a small percentage of “incorrect” amino acids may continue to function. However, it rarely functions as well as a protein having the correct sequence. There are also instances in which seemingly minor errors of sequence have disastrous effects. For example, in some people, every molecule of hemoglobin (a protein in the blood that transports oxygen) has a single incorrect amino acid unit out of about 300 (a single valine replaces a glutamic acid). That “minor” error is responsible for sickle cell anemia, an inherited condition that usually is fatal.</p>
    <div class="qandaset block" id="gob-ch18_s03_qs01" condition="1col" defaultlabel="number">
        <h3 class="title">Concept Review Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s03_qs01_qd01">
            <li class="qandaentry" id="gob-ch18_s03_qs01_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s03_qs01_p01">Distinguish between the N-terminal amino acid and the C-terminal amino acid of a peptide or protein.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs01_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s03_qs01_p03">Describe the difference between an amino acid and a peptide.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs01_qd01_qa03">
                <div class="question">
                    <p class="para" id="gob-ch18_s03_qs01_p05">Amino acid units in a protein are connected by peptide bonds. What is another name for the functional group linking the amino acids?</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s03_qs01_ans" condition="1col" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s03_qs01_qd01_qa01_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s03_qs01_p02_ans">The N-terminal end is the end of a peptide or protein whose amino group is free (not involved in the formation of a peptide bond), while the C-terminal end has a free carboxyl group.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs01_qd01_qa02_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s03_qs01_p04_ans">A peptide is composed of two or more amino acids. Amino acids are the building blocks of peptides.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs01_qd01_qa03_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s03_qs01_p06_ans">amide bond</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="key_takeaways editable block" id="gob-ch18_s03_n03">
        <h3 class="title">Key Takeaways</h3>
        <ul class="itemizedlist" id="gob-ch18_s03_l03">
            <li>The amino group of one amino acid can react with the carboxyl group on another amino acid to form a peptide bond that links the two amino acids together. Additional amino acids can be added on through the formation of addition peptide (amide) bonds.</li>
            <li>A sequence of amino acids in a peptide or protein is written with the N-terminal amino acid first and the C-terminal amino acid at the end (writing left to right).</li>
        </ul>
    </div>
    <div class="qandaset block" id="gob-ch18_s03_qs02" condition="1col" defaultlabel="number">
        <h3 class="title">Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s03_qs02_qd01">
            <li class="qandaentry" id="gob-ch18_s03_qs02_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s03_qs02_p01">Draw the structure for each peptide.</p>
                    <ol class="orderedlist" id="gob-ch18_s03_qs02_l01" numeration="loweralpha">
                        <li>gly-val</li>
                        <li>val-gly</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs02_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s03_qs02_p02">Draw the structure for cys-val-ala.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs02_qd01_qa03">
                <div class="question">
                    <p class="para" id="gob-ch18_s03_qs02_p03">Identify the C- and N-terminal amino acids for the peptide lys-val-phe-gly-arg-cys.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs02_qd01_qa04">
                <div class="question">
                    <p class="para" id="gob-ch18_s03_qs02_p05">Identify the C- and N-terminal amino acids for the peptide asp-arg-val-tyr-ile-his-pro-phe.</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s03_qs02_ans" condition="1col" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s03_qs02_qd01_qa01_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s03_qs02_l02_ans" numeration="loweralpha">                            <li>
                                <div class="informalfigure large" condition="equation-image">
                                    <img src="section_21/11673f1d38b19a90480d2e4f44636525.jpg">
                                </div>
                            </li>
                            <li>
                                <div class="informalfigure large" condition="equation-image">
                                    <img src="section_21/8e5098390ccb1ec2ad6ae60f20028300.jpg">
                                </div>
                            </li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs02_qd01_qa02_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs02_qd01_qa03_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s03_qs02_p04_ans">C-terminal amino acid: cys; N-terminal amino acid: lys</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s03_qs02_qd01_qa04_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
        </ol>
    </div>
</div>
<div class="section" id="gob-ch18_s04" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.4</span> Proteins</h2>
    <div class="learning_objectives editable block" id="gob-ch18_s04_n01">
        <h3 class="title">Learning Objectives</h3>
        <ol class="orderedlist" id="gob-ch18_s04_l01">
            <li>Describe the four levels of protein structure.</li>
            <li>Identify the types of attractive interactions that hold proteins in their most stable three-dimensional structure.</li>
            <li>Explain what happens when proteins are denatured.</li>
            <li>Identify how a protein can be denatured.</li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s04_p01">Each of the thousands of naturally occurring proteins has its own characteristic amino acid composition and sequence that result in a unique three-dimensional shape. Since the 1950s, scientists have determined the amino acid sequences and three-dimensional conformation of numerous proteins and thus obtained important clues on how each protein performs its specific function in the body.</p>
    <p class="para editable block" id="gob-ch18_s04_p02">Proteins are compounds of high molar mass consisting largely or entirely of chains of amino acids. Because of their great complexity, protein molecules cannot be classified on the basis of specific structural similarities, as carbohydrates and lipids are categorized. The two major structural classifications of proteins are based on far more general qualities: whether the protein is (1) fiberlike and insoluble or (2) globular and soluble. Some proteins, such as those that compose hair, skin, muscles, and connective tissue, are fiberlike. These <span class="margin_term"><a class="glossterm">fibrous proteins</a><span class="glossdef">A protein that is elongated or fiberlike and insoluble in water.</span></span> are insoluble in water and usually serve structural, connective, and protective functions. Examples of fibrous proteins are keratins, collagens, myosins, and elastins. Hair and the outer layer of skin are composed of keratin. Connective tissues contain collagen. Myosins are muscle proteins and are capable of contraction and extension. Elastins are found in ligaments and the elastic tissue of artery walls.</p>
    <p class="para editable block" id="gob-ch18_s04_p03"><span class="margin_term"><a class="glossterm">Globular proteins</a><span class="glossdef">A protein that is generally spherical in structure and soluble in water.</span></span>, the other major class, are soluble in aqueous media. In these proteins, the chains are folded so that the molecule as a whole is roughly spherical. Familiar examples include egg albumin from egg whites and serum albumin in blood. Serum albumin plays a major role in transporting fatty acids and maintaining a proper balance of osmotic pressures in the body. Hemoglobin and myoglobin, which are important for binding oxygen, are also globular proteins.</p>
    <div class="section" id="gob-ch18_s04_s01">
        <h2 class="title editable block">Levels of Protein Structure</h2>
        <p class="para editable block" id="gob-ch18_s04_s01_p01">The structure of proteins is generally described as having four organizational levels. The first of these is the <span class="margin_term"><a class="glossterm">primary structure</a><span class="glossdef">The sequence of amino acids in a polypeptide chain or protein.</span></span>, which is the number and sequence of amino acids in a protein’s polypeptide chain or chains, beginning with the free amino group and maintained by the peptide bonds connecting each amino acid to the next. The primary structure of insulin, composed of 51 amino acids, is shown in <a class="xref" href="#gob-ch18_s04_s01_f01">Figure 18.2 "Primary Structure of Human Insulin"</a>.</p>
        <div class="figure large editable block" id="gob-ch18_s04_s01_f01">
            <p class="title"><span class="title-prefix">Figure 18.2</span> Primary Structure of Human Insulin</p>
            <img src="section_21/1c0cf9b4785df1547a24bd9c3a27a867.jpg">
            <p class="para">Human insulin, whose amino acid sequence is shown here, is a hormone that is required for the proper metabolism of glucose.</p>
        </div>
        <p class="para editable block" id="gob-ch18_s04_s01_p02">A protein molecule is not a random tangle of polypeptide chains. Instead, the chains are arranged in unique but specific conformations. The term <span class="margin_term"><a class="glossterm">secondary structure</a><span class="glossdef">The fixed arrangement of the polypeptide backbone.</span></span> refers to the fixed arrangement of the polypeptide backbone. On the basis of X ray studies, Linus Pauling and Robert Corey postulated that certain proteins or portions of proteins twist into a spiral or a helix. This helix is stabilized by <em class="emphasis">intrachain</em> hydrogen bonding between the carbonyl oxygen atom of one amino acid and the amide hydrogen atom four amino acids up the chain (located on the next turn of the helix) and is known as a right-handed α-helix. X ray data indicate that this helix makes one turn for every 3.6 amino acids, and the side chains of these amino acids project outward from the coiled backbone (<a class="xref" href="#gob-ch18_s04_s01_f02">Figure 18.3 "A Ball-and-Stick Model of an α-Helix"</a>). The α-keratins, found in hair and wool, are exclusively α-helical in conformation. Some proteins, such as gamma globulin, chymotrypsin, and cytochrome c, have little or no helical structure. Others, such as hemoglobin and myoglobin, are helical in certain regions but not in others.</p>
        <div class="figure large medium-height editable block" id="gob-ch18_s04_s01_f02">
            <p class="title"><span class="title-prefix">Figure 18.3</span> A Ball-and-Stick Model of an α-Helix</p>
            <img src="section_21/79bb1c5101fce9ea91eaa84016a73e2f.jpg">
            <p class="para">This ball-and-stick model shows the intrachain hydrogen bonding between carbonyl oxygen atoms and amide hydrogen atoms. Each turn of the helix spans 3.6 amino acids. Note that the side chains (represented as green spheres) point out from the helix.</p>
        </div>
        <p class="para editable block" id="gob-ch18_s04_s01_p03">Another common type of secondary structure, called the <em class="emphasis">β-pleated sheet conformation</em>, is a sheetlike arrangement in which two or more extended polypeptide chains (or separate regions on the same chain) are aligned side by side. The aligned segments can run either parallel or antiparallel—that is, the N-terminals can face in the same direction on adjacent chains or in different directions—and are connected by <em class="emphasis">interchain</em> hydrogen bonding (<a class="xref" href="#gob-ch18_s04_s01_f03">Figure 18.4 "A Ball-and-Stick Model of the β-Pleated Sheet Structure in Proteins"</a>). The β-pleated sheet is particularly important in structural proteins, such as silk fibroin. It is also seen in portions of many enzymes, such as carboxypeptidase A and lysozyme.</p>
        <div class="figure large editable block" id="gob-ch18_s04_s01_f03">
            <p class="title"><span class="title-prefix">Figure 18.4</span> A Ball-and-Stick Model of the β-Pleated Sheet Structure in Proteins</p>
            <img src="section_21/51229446d0888134f4a9856b8213c864.jpg">
            <p class="para">The side chains extend above or below the sheet and alternate along the chain. The protein chains are held together by interchain hydrogen bonding.</p>
        </div>
        <p class="para editable block" id="gob-ch18_s04_s01_p04"><span class="margin_term"><a class="glossterm">Tertiary structure</a><span class="glossdef">The unique three-dimensional shape of a polypeptide chain as a whole.</span></span> refers to the unique three-dimensional shape of the protein as a whole, which results from the folding and bending of the protein backbone. The tertiary structure is intimately tied to the proper biochemical functioning of the protein. <a class="xref" href="#gob-ch18_s04_s01_f04">Figure 18.5 "A Ribbon Model of the Three-Dimensional Structure of Insulin"</a> shows a depiction of the three-dimensional structure of insulin.</p>
        <div class="figure large medium-height editable block" id="gob-ch18_s04_s01_f04">
            <p class="title"><span class="title-prefix">Figure 18.5</span> A Ribbon Model of the Three-Dimensional Structure of Insulin</p>
            <img src="section_21/cc0d7b0cf19a3f672b5deba0fc56b44f.jpg">
            <p class="para">The spiral regions represent sections of the polypeptide chain that have an α-helical structure, while the broad arrows represent β-pleated sheet structures.</p>
        </div>
        <p class="para editable block" id="gob-ch18_s04_s01_p05">Four major types of attractive interactions determine the shape and stability of the tertiary structure of proteins. You studied several of these in <a class="xref" href="gob-ch08#gob-ch08">Chapter 8 "Solids, Liquids, and Gases"</a>, <a class="xref" href="gob-ch08_s01#gob-ch08_s01">Section 8.1 "Intermolecular Interactions"</a>.</p>
        <p class="para editable block" id="gob-ch18_s04_s01_p06">1.	<span class="margin_term"><a class="glossterm">Ionic bonding</a><span class="glossdef">Bonding that results from electrostatic attractions between positively and negatively charged groups.</span></span>. Ionic bonds result from electrostatic attractions between positively and negatively charged side chains of amino acids. For example, the mutual attraction between an aspartic acid carboxylate ion and a lysine ammonium ion helps to maintain a particular folded area of a protein (part (a) of <a class="xref" href="#gob-ch18_s04_s01_f05">Figure 18.6 "Tertiary Protein Structure Interactions"</a>).</p>
        <p class="para editable block" id="gob-ch18_s04_s01_p07">2.	<span class="margin_term"><a class="glossterm">Hydrogen bonding</a><span class="glossdef">Bonding between a highly electronegative oxygen atom or nitrogen atom and a hydrogen atom attached to another oxygen atom or nitrogen atom.</span></span>. Hydrogen bonding forms between a highly electronegative oxygen atom or a nitrogen atom and a hydrogen atom attached to another oxygen atom or a nitrogen atom, such as those found in polar amino acid side chains. Hydrogen bonding (as well as ionic attractions) is extremely important in both the intra- and intermolecular interactions of proteins (part (b) of <a class="xref" href="#gob-ch18_s04_s01_f05">Figure 18.6 "Tertiary Protein Structure Interactions"</a>).</p>
        <p class="para editable block" id="gob-ch18_s04_s01_p08">3.	<span class="margin_term"><a class="glossterm">Disulfide linkages</a><span class="glossdef">A covalent bond that forms by the oxidation and linkage of two sulfur atoms from the side chains of two cysteine residues.</span></span>. Two cysteine amino acid units may be brought close together as the protein molecule folds. Subsequent oxidation and linkage of the sulfur atoms in the highly reactive sulfhydryl (SH) groups leads to the formation of cystine (part (c) of <a class="xref" href="#gob-ch18_s04_s01_f05">Figure 18.6 "Tertiary Protein Structure Interactions"</a>). Intrachain disulfide linkages are found in many proteins, including insulin (yellow bars in <a class="xref" href="#gob-ch18_s04_s01_f01">Figure 18.2 "Primary Structure of Human Insulin"</a>) and have a strong stabilizing effect on the tertiary structure.</p>
        <div class="informalfigure large block">
            <img src="section_21/c02399a4fba322ebf86aa960bd158f0d.jpg">
        </div>
        <p class="para editable block" id="gob-ch18_s04_s01_p09">4.	<span class="margin_term"><a class="glossterm">Dispersion forces</a><span class="glossdef">A force caused by the instantaneous imbalance of electrons about a molecule.</span></span>. Dispersion forces arise when a normally nonpolar atom becomes momentarily polar due to an uneven distribution of electrons, leading to an instantaneous dipole that induces a shift of electrons in a neighboring nonpolar atom. Dispersion forces are weak but can be important when other types of interactions are either missing or minimal (part (d) of <a class="xref" href="#gob-ch18_s04_s01_f05">Figure 18.6 "Tertiary Protein Structure Interactions"</a>). This is the case with fibroin, the major protein in silk, in which a high proportion of amino acids in the protein have nonpolar side chains. The term <em class="emphasis">hydrophobic interaction</em> is often misused as a synonym for dispersion forces. Hydrophobic interactions arise because water molecules engage in hydrogen bonding with other water molecules (or groups in proteins capable of hydrogen bonding). Because nonpolar groups cannot engage in hydrogen bonding, the protein folds in such a way that these groups are buried in the interior part of the protein structure, minimizing their contact with water.</p>
        <div class="figure large editable block" id="gob-ch18_s04_s01_f05">
            <p class="title"><span class="title-prefix">Figure 18.6</span> Tertiary Protein Structure Interactions</p>
            <img src="section_21/4bcc36743f97d8194661279d48f87367.jpg">
            <p class="para">Four interactions stabilize the tertiary structure of a protein: (a) ionic bonding, (b) hydrogen bonding, (c) disulfide linkages, and (d) dispersion forces.</p>
        </div>
        <p class="para editable block" id="gob-ch18_s04_s01_p10">When a protein contains more than one polypeptide chain, each chain is called a <em class="emphasis">subunit</em>. The arrangement of multiple subunits represents a fourth level of structure, the <span class="margin_term"><a class="glossterm">quaternary structure</a><span class="glossdef">The arrangement of multiple subunits in a protein.</span></span> of a protein. Hemoglobin, with four polypeptide chains or subunits, is the most frequently cited example of a protein having quaternary structure (<a class="xref" href="#gob-ch18_s04_s01_f06">Figure 18.7 "The Quaternary Structure of Hemoglobin"</a>). The quaternary structure of a protein is produced and stabilized by the same kinds of interactions that produce and maintain the tertiary structure. A schematic representation of the four levels of protein structure is in <a class="xref" href="#gob-ch18_s04_s01_f07">Figure 18.8 "Levels of Structure in Proteins"</a>.</p>
        <div class="figure large medium-height editable block" id="gob-ch18_s04_s01_f06">
            <p class="title"><span class="title-prefix">Figure 18.7</span> The Quaternary Structure of Hemoglobin</p>
            <img src="section_21/d7a8c2e47f3a05747b90791c7a2485be.jpg">
            <p class="para">Hemoglobin is a protein that transports oxygen throughout the body.</p>
            <div class="copyright">
                <p class="para">Source: Image from the RCSB PDB (<a class="link" target="_blank" href="http://www.pdb.org">www.pdb.org</a>) of PDB ID 1I3D (R.D. Kidd, H.M. Baker, A.J. Mathews, T. Brittain, E.N. Baker (2001) Oligomerization and ligand binding in a homotetrameric hemoglobin: two high-resolution crystal structures of hemoglobin Bart's (gamma(4)), a marker for alpha-thalassemia. Protein Sci. 1739–1749).</p>
            </div>
        </div>
        <div class="figure large editable block" id="gob-ch18_s04_s01_f07">
            <p class="title"><span class="title-prefix">Figure 18.8</span> Levels of Structure in Proteins</p>
            <img src="section_21/6d9b4af09c98e66f071eb9560fd984ef.jpg">
            <p class="para">The <em class="emphasis">primary structure</em> consists of the specific amino acid sequence. The resulting peptide chain can twist into an α-helix, which is one type of <em class="emphasis">secondary structure</em>. This helical segment is incorporated into the <em class="emphasis">tertiary structure</em> of the folded polypeptide chain. The single polypeptide chain is a subunit that constitutes the <em class="emphasis">quaternary structure</em> of a protein, such as hemoglobin that has four polypeptide chains.</p>
        </div>
    </div>
    <div class="section" id="gob-ch18_s04_s02">
        <h2 class="title editable block">Denaturation of Proteins</h2>
        <p class="para editable block" id="gob-ch18_s04_s02_p01">The highly organized structures of proteins are truly masterworks of chemical architecture. But highly organized structures tend to have a certain delicacy, and this is true of proteins. <span class="margin_term"><a class="glossterm">Denaturation</a><span class="glossdef">Any change in the three-dimensional structure of a macromolecule that renders it incapable of performing its assigned function.</span></span> is the term used for any change in the three-dimensional structure of a protein that renders it incapable of performing its assigned function. A denatured protein cannot do its job. (Sometimes denaturation is equated with the precipitation or coagulation of a protein; our definition is a bit broader.) A wide variety of reagents and conditions, such as heat, organic compounds, pH changes, and heavy metal ions can cause protein denaturation (<a class="xref" href="#gob-ch18_s04_s02_t01">Table 18.4 "Protein Denaturation Methods"</a>).</p>
        <div class="table block" id="gob-ch18_s04_s02_t01" frame="all">
            <p class="title"><span class="title-prefix">Table 18.4</span> Protein Denaturation Methods</p>
            <table cellpadding="0" cellspacing="0">
                <thead>
                    <tr>
                        <th>Method</th>
                        <th>Effect on Protein Structure</th>
                    </tr>
                </thead>
                <tbody>
                    <tr>
                        <td>Heat above 50°C or ultraviolet (UV) radiation</td>
                        <td>Heat or UV radiation supplies kinetic energy to protein molecules, causing their atoms to vibrate more rapidly and disrupting relatively weak hydrogen bonding and dispersion forces.</td>
                    </tr>
                    <tr>
                        <td>Use of organic compounds, such as ethyl alcohol</td>
                        <td>These compounds are capable of engaging in intermolecular hydrogen bonding with protein molecules, disrupting intramolecular hydrogen bonding within the protein.</td>
                    </tr>
                    <tr>
                        <td>Salts of heavy metal ions, such as mercury, silver, and lead</td>
                        <td>These ions form strong bonds with the carboxylate anions of the acidic amino acids or SH groups of cysteine, disrupting ionic bonds and disulfide linkages.</td>
                    </tr>
                    <tr>
                        <td>Alkaloid reagents, such as tannic acid (used in tanning leather)</td>
                        <td>These reagents combine with positively charged amino groups in proteins to disrupt ionic bonds.</td>
                    </tr>
                </tbody>
            </table>
        </div>
        <p class="para editable block" id="gob-ch18_s04_s02_p02">Anyone who has fried an egg has observed denaturation. The clear egg white turns opaque as the albumin denatures and coagulates. No one has yet reversed that process. However, given the proper circumstances and enough time, a protein that has unfolded under sufficiently gentle conditions can refold and may again exhibit biological activity (<a class="xref" href="#gob-ch18_s04_s02_f01">Figure 18.9 "Denaturation and Renaturation of a Protein"</a>). Such evidence suggests that, at least for these proteins, the primary structure determines the secondary and tertiary structure. A given sequence of amino acids seems to adopt its particular three-dimensional arrangement naturally if conditions are right.</p>
        <div class="figure large medium-height editable block" id="gob-ch18_s04_s02_f01">
            <p class="title"><span class="title-prefix">Figure 18.9</span> Denaturation and Renaturation of a Protein</p>
            <img src="section_21/9e024c11709a6bb80a04338d399c3a20.jpg">
            <p class="para">The denaturation (unfolding) and renaturation (refolding) of a protein is depicted. The red boxes represent stabilizing interactions, such as disulfide linkages, hydrogen bonding, and/or ionic bonds.</p>
        </div>
        <p class="para editable block" id="gob-ch18_s04_s02_p03">The primary structures of proteins are quite sturdy. In general, fairly vigorous conditions are needed to hydrolyze peptide bonds. At the secondary through quaternary levels, however, proteins are quite vulnerable to attack, though they vary in their vulnerability to denaturation. The delicately folded globular proteins are much easier to denature than are the tough, fibrous proteins of hair and skin.</p>
        <div class="qandaset block" id="gob-ch18_s04_qs01" condition="1col" defaultlabel="number">
            <h3 class="title">Concept Review Exercises</h3>
            <ol class="qandadiv" id="gob-ch18_s04_qs01_qd01">
                <li class="qandaentry" id="gob-ch18_s04_qs01_qd01_qa01">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs01_p01">What is the predominant attractive force that stabilizes the formation of secondary structure in proteins?</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs01_qd01_qa02">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs01_p03">Distinguish between the tertiary and quaternary levels of protein structure.</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs01_qd01_qa03">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs01_p05">Briefly describe four ways in which a protein could be denatured.</p>
                    </div>
                </li>
            </ol>
        </div>
        <div class="qandaset block" id="gob-ch18_s04_qs01_ans" condition="1col" defaultlabel="number">
            <h3 class="title">Answers</h3>
            <ol class="qandadiv">
                <li class="qandaentry" id="gob-ch18_s04_qs01_qd01_qa01_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s04_qs01_p02_ans">hydrogen bonding</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs01_qd01_qa02_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s04_qs01_p04_ans">Tertiary structure refers to the unique three-dimensional shape of a single polypeptide chain, while quaternary structure describes the interaction between multiple polypeptide chains for proteins that have more than one polypeptide chain.</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs01_qd01_qa03_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s04_qs01_p06_ans">(1) heat a protein above 50°C or expose it to UV radiation; (2) add organic solvents, such as ethyl alcohol, to a protein solution; (3) add salts of heavy metal ions, such as mercury, silver, or lead; and (4) add alkaloid reagents such as tannic acid</p>
                    </div>
                </li>
            </ol>
        </div>
        <div class="key_takeaways editable block" id="gob-ch18_s04_s02_n02">
            <h3 class="title">Key Takeaways</h3>
            <ul class="itemizedlist" id="gob-ch18_s04_s02_l02">
                <li>Proteins can be divided into two categories: fibrous, which tend to be insoluble in water, and globular, which are more soluble in water.</li>
                <li>A protein may have up to four levels of structure. The primary structure consists of the specific amino acid sequence. The resulting peptide chain can form an α-helix or β-pleated sheet (or local structures not as easily categorized), which is known as secondary structure. These segments of secondary structure are incorporated into the tertiary structure of the folded polypeptide chain. The quaternary structure describes the arrangements of subunits in a protein that contains more than one subunit.</li>
                <li>Four major types of attractive interactions determine the shape and stability of the folded protein: ionic bonding, hydrogen bonding, disulfide linkages, and dispersion forces.</li>
                <li>A wide variety of reagents and conditions can cause a protein to unfold or denature.</li>
            </ul>
        </div>
        <div class="qandaset block" id="gob-ch18_s04_qs02" defaultlabel="number">
            <h3 class="title">Exercises</h3>
            <ol class="qandadiv" id="gob-ch18_s04_qs02_qd01">
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa01">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs02_p01">Classify each protein as fibrous or globular.</p>
                        <ol class="orderedlist" id="gob-ch18_s04_qs02_l01" numeration="loweralpha">
                            <li>albumin</li>
                            <li>myosin</li>
                            <li>fibroin</li>
                        </ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa02">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs02_p02">Classify each protein as fibrous or globular.</p>
                        <ol class="orderedlist" id="gob-ch18_s04_qs02_l03" numeration="loweralpha">
                            <li>hemoglobin</li>
                            <li>keratin</li>
                            <li>myoglobin</li>
                        </ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa03">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs02_p03">What name is given to the predominant secondary structure found in silk?</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa04">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs02_p05">What name is given to the predominant secondary structure found in wool protein?</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa05">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs02_p07">A protein has a tertiary structure formed by interactions between the side chains of the following pairs of amino acids. For each pair, identify the strongest type of interaction between these amino acids.</p>
                        <ol class="orderedlist" id="gob-ch18_s04_qs02_l05" numeration="loweralpha">
                            <li>aspartic acid and lysine</li>
                            <li>phenylalanine and alanine</li>
                            <li>serine and lysine</li>
                            <li>two cysteines</li>
                        </ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa06">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs02_p08">A protein has a tertiary structure formed by interactions between the side chains of the following pairs of amino acids. For each pair, identify the strongest type of interaction between these amino acids.</p>
                        <ol class="orderedlist" id="gob-ch18_s04_qs02_l07" numeration="loweralpha">
                            <li>valine and isoleucine</li>
                            <li>asparagine and serine</li>
                            <li>glutamic acid and arginine</li>
                            <li>tryptophan and methionine</li>
                        </ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa07">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs02_p09">What level(s) of protein structure is(are) ordinarily disrupted in denaturation? What level(s) is(are) not?</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa08">
                    <div class="question">
                        <p class="para" id="gob-ch18_s04_qs02_p11">Which class of proteins is more easily denatured—fibrous or globular?</p>
                    </div>
                </li>
            </ol>
        </div>
        <div class="qandaset block" id="gob-ch18_s04_qs02_ans" defaultlabel="number">
            <h3 class="title">Answers</h3>
            <ol class="qandadiv">
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa01_ans">
                    <div class="answer">
                        <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s04_qs02_l02_ans" numeration="loweralpha">                                <li>globular</li>
                                <li>fibrous</li>
                                <li>fibrous</li>
</ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa02_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa03_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s04_qs02_p04_ans">β-pleated sheet</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa04_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa05_ans">
                    <div class="answer">
                        <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s04_qs02_l06_ans" numeration="loweralpha">                                <li>ionic bonding</li>
                                <li>dispersion forces</li>
                                <li>dispersion forces</li>
                                <li>disulfide linkage</li>
</ol>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa06_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa07_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s04_qs02_p10_ans">Protein denaturation disrupts the secondary, tertiary, and quaternary levels of structure. Only primary structure is unaffected by denaturation.</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s04_qs02_qd01_qa08_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
            </ol>
        </div>
    </div>
</div>
<div class="section" id="gob-ch18_s05" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.5</span> Enzymes</h2>
    <div class="learning_objectives editable block" id="gob-ch18_s05_n01">
        <h3 class="title">Learning Objectives</h3>
        <ol class="orderedlist" id="gob-ch18_s05_l01">
            <li>Explain the functions of enzymes.</li>
            <li>Explain how enzymes are classified and named.</li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s05_p01">A <span class="margin_term"><a class="glossterm">catalyst</a><span class="glossdef">Any substance that increases the rate or speed of a chemical reaction without being changed or consumed in the reaction.</span></span> is any substance that increases the <em class="emphasis">rate</em> or speed of a chemical reaction without being changed or consumed in the reaction. <span class="margin_term"><a class="glossterm">Enzymes</a><span class="glossdef">A biological catalyst.</span></span> are biological catalysts, and nearly all of them are proteins. The reaction rates attained by enzymes are truly amazing. In their presence, reactions occur at rates that are a million (10<sup class="superscript">6</sup>) or more times faster than would be attainable in their absence. What is even more amazing is that enzymes perform this function at body temperature (~37°C) and physiological pH (pH ~7), rather than at the conditions that are typically necessary to increase reaction rates (high temperature or pressure, the use of strong oxidizing or reducing agents or strong acids or bases, or a combination of any of these). In addition, enzymes are highly specific in their action; that is, each enzyme catalyzes only one type of reaction in only one compound or a group of structurally related compounds. The compound or compounds on which an enzyme acts are known as its <span class="margin_term"><a class="glossterm">substrates</a><span class="glossdef">A compound on which an enzyme acts.</span></span>.</p>
    <p class="para editable block" id="gob-ch18_s05_p02">Hundreds of enzymes have been purified and studied in an effort to understand how they work so effectively and with such specificity. The resulting knowledge has been used to design drugs that inhibit or activate particular enzymes. An example is the intensive research to improve the treatment of or find a cure for acquired immunodeficiency syndrome (AIDS). AIDS is caused by the human immunodeficiency virus (HIV). Researchers are studying the enzymes produced by this virus and are developing drugs intended to block the action of those enzymes without interfering with enzymes produced by the human body. Several of these drugs have now been approved for use by AIDS patients.</p>
    <p class="para editable block" id="gob-ch18_s05_p03">The first enzymes to be discovered were named according to their source or method of discovery. The enzyme <em class="emphasis">pepsin</em>, which aids in the hydrolysis of proteins, is found in the digestive juices of the stomach (Greek <em class="emphasis">pepsis</em>, meaning “digestion”). <em class="emphasis">Papain</em>, another enzyme that hydrolyzes protein (in fact, it is used in meat tenderizers), is isolated from papayas. As more enzymes were discovered, chemists recognized the need for a more systematic and chemically informative identification scheme.</p>
    <p class="para editable block" id="gob-ch18_s05_p04">In the current numbering and naming scheme, under the oversight of the Nomenclature Commission of the International Union of Biochemistry, enzymes are arranged into six groups according to the general type of reaction they catalyze (<a class="xref" href="#gob-ch18_s05_t01">Table 18.5 "Classes of Enzymes"</a>), with subgroups and secondary subgroups that specify the reaction more precisely. Each enzyme is assigned a four-digit number, preceded by the prefix EC—for enzyme classification—that indicates its group, subgroup, and so forth. This is demonstrated in <a class="xref" href="#gob-ch18_s05_t02">Table 18.6 "Assignment of an Enzyme Classification Number"</a> for alcohol dehydrogenase. Each enzyme is also given a name consisting of the root of the name of its substrate or substrates and the -<em class="emphasis">ase</em> suffix. Thus urease is the enzyme that catalyzes the hydrolysis of urea.</p>
    <div class="table block" id="gob-ch18_s05_t01" frame="all">
        <p class="title"><span class="title-prefix">Table 18.5</span> Classes of Enzymes</p>
        <table cellpadding="0" cellspacing="0">
            <thead>
                <tr>
                    <th>Class</th>
                    <th>Type of Reaction Catalyzed</th>
                    <th>Examples</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>oxidoreductases</td>
                    <td>oxidation-reduction reactions</td>
                    <td>Dehydrogenases catalyze oxidation-reduction reactions involving hydrogen and reductases catalyze reactions in which a substrate is reduced.</td>
                </tr>
                <tr>
                    <td>transferases</td>
                    <td>transfer reactions of groups, such as methyl, amino, and acetyl</td>
                    <td>Transaminases catalyze the transfer of amino group, and kinases catalyze the transfer of a phosphate group.</td>
                </tr>
                <tr>
                    <td>hydrolases</td>
                    <td>hydrolysis reactions</td>
                    <td>Lipases catalyze the hydrolysis of lipids, and proteases catalyze the hydrolysis of proteins</td>
                </tr>
                <tr>
                    <td>lyases</td>
                    <td>reactions in which groups are removed without hydrolysis or addition of groups to a double bond</td>
                    <td>Decarboxylases catalyze the removal of carboxyl groups.</td>
                </tr>
                <tr>
                    <td>isomerases</td>
                    <td>reactions in which a compound is converted to its isomer</td>
                    <td>Isomerases may catalyze the conversion of an aldose to a ketose, and mutases catalyze reactions in which a functional group is transferred from one atom in a substrate to another.</td>
                </tr>
                <tr>
                    <td>ligases</td>
                    <td>reactions in which new bonds are formed between carbon and another atom; energy is required</td>
                    <td>Synthetases catalyze reactions in which two smaller molecules are linked to form a larger one.</td>
                </tr>
            </tbody>
        </table>
    </div>
    <div class="table block" id="gob-ch18_s05_t02" frame="all">
        <p class="title"><span class="title-prefix">Table 18.6</span> Assignment of an Enzyme Classification Number</p>
        <table cellpadding="0" cellspacing="0">
            <thead>
                <tr>
                    <th colspan="2">Alcohol Dehydrogenase: EC 1.1.1.1</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td colspan="2">The first digit indicates that this enzyme is an oxidoreductase; that is, an enzyme that catalyzes an oxidation-reduction reaction.</td>
                </tr>
                <tr>
                    <td colspan="2">The second digit indicates that this oxidoreductase catalyzes a reaction involving a primary or secondary alcohol.</td>
                </tr>
                <tr>
                    <td colspan="2">The third digit indicates that either the coenzyme NAD<sup class="superscript">+</sup> or NADP<sup class="superscript">+</sup> is required for this reaction.</td>
                </tr>
                <tr>
                    <td colspan="2">The fourth digit indicates that this was the first enzyme isolated, characterized, and named using this system of nomenclature.</td>
                </tr>
                <tr>
                    <td colspan="2">The systematic name for this enzyme is <em class="emphasis">alcohol:NAD</em><sup class="superscript">+</sup> <em class="emphasis">oxidoreductase</em>, while the recommended or common name is alcohol dehydrogenase.</td>
                </tr>
                <tr>
                    <td><p class="para">Reaction catalyzed:</p></td>
                    <td>
	            <div class="informalfigure medium">
                <img src="section_21/d7cba09ae63bb89a4d3fcc32a4fe15a6.jpg">
            </div>
</td>
                </tr>
            </tbody>
        </table>
    </div>
    <div class="qandaset block" id="gob-ch18_s05_qs01" condition="1col" defaultlabel="number">
        <h3 class="title">Concept Review Exercise</h3>
        <ol class="qandadiv" id="gob-ch18_s05_qs01_qd01">
            <p class="para" id="gob-ch18_s05_qs01_p01">In the small intestine, sucrose is hydrolyzed to form glucose and fructose in a reaction catalyzed by sucrase.</p>
            <li class="qandaentry" id="gob-ch18_s05_qs01_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s05_qs01_p02">Identify the substrate in this reaction.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s05_qs01_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s05_qs01_p04">Name the enzyme.</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s05_qs01_ans" condition="1col" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s05_qs01_qd01_qa01_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s05_qs01_p03_ans">sucrose</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s05_qs01_qd01_qa02_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s05_qs01_p05_ans">sucrase</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="key_takeaways editable block" id="gob-ch18_s05_n03">
        <h3 class="title">Key Takeaways</h3>
        <ul class="itemizedlist" id="gob-ch18_s05_l03">
            <li>An enzyme is a biological catalyst, a substance that increases the rate of a chemical reaction without being changed or consumed in the reaction.</li>
            <li>A systematic process is used to name and classify enzymes.</li>
        </ul>
    </div>
    <div class="qandaset block" id="gob-ch18_s05_qs02" defaultlabel="number">
        <h3 class="title">Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s05_qs02_qd01">
            <li class="qandaentry" id="gob-ch18_s05_qs02_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s05_qs02_p01">Identify the substrate catalyzed by each enzyme.</p>
                    <ol class="orderedlist" id="gob-ch18_s05_qs02_l01" numeration="loweralpha">
                        <li>lactase</li>
                        <li>cellulase</li>
                        <li>peptidase</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s05_qs02_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s05_qs02_p02">Identify the substrate catalyzed by each enzyme.</p>
                    <ol class="orderedlist" id="gob-ch18_s05_qs02_l03" numeration="loweralpha">
                        <li>lipase</li>
                        <li>amylase</li>
                        <li>maltase</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s05_qs02_qd01_qa03">
                <div class="question">
                    <p class="para" id="gob-ch18_s05_qs02_p03">Identify each type of enzyme.</p>
                    <ol class="orderedlist" id="gob-ch18_s05_qs02_l05" numeration="loweralpha">
                        <li>decarboxylase</li>
                        <li>protease</li>
                        <li>transaminase</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s05_qs02_qd01_qa04">
                <div class="question">
                    <p class="para" id="gob-ch18_s05_qs02_p04">Identify each type of enzyme.</p>
                    <ol class="orderedlist" id="gob-ch18_s05_qs02_l07" numeration="loweralpha">
                        <li>dehydrogenase</li>
                        <li>isomerase</li>
                        <li>lipase</li>
                    </ol>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s05_qs02_ans" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s05_qs02_qd01_qa01_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s05_qs02_l02_ans" numeration="loweralpha">                            <li>lactose</li>
                            <li>cellulose</li>
                            <li>peptides</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s05_qs02_qd01_qa02_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s05_qs02_qd01_qa03_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s05_qs02_l06_ans" numeration="loweralpha">                            <li>lyase</li>
                            <li>hydrolase</li>
                            <li>transferase</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s05_qs02_qd01_qa04_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
        </ol>
    </div>
</div>
<div class="section" id="gob-ch18_s06" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.6</span> Enzyme Action</h2>
    <div class="learning_objectives editable block" id="gob-ch18_s06_n01">
        <h3 class="title">Learning Objective</h3>
        <ol class="orderedlist" id="gob-ch18_s06_l01">
            <li>Describe the interaction between an enzyme and its substrate.</li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s06_p01">Enzyme-catalyzed reactions occur in at least two steps. In the first step, an enzyme molecule (E) and the substrate molecule or molecules (S) collide and react to form an intermediate compound called the <em class="emphasis">enzyme-substrate</em> (E–S) <em class="emphasis">complex</em>. (This step is reversible because the complex can break apart into the original substrate or substrates and the free enzyme.) Once the E–S complex forms, the enzyme is able to catalyze the formation of product (P), which is then released from the enzyme surface:</p>
    <span class="informalequation block"><span class="mathphrase">S + E → E–S</span></span>
    <span class="informalequation block"><span class="mathphrase">E–S → P + E</span></span>
    <p class="para editable block" id="gob-ch18_s06_p02">Hydrogen bonding and other electrostatic interactions hold the enzyme and substrate together in the complex. The structural features or functional groups on the enzyme that participate in these interactions are located in a cleft or pocket on the enzyme surface. This pocket, where the enzyme combines with the substrate and transforms the substrate to product is called the <span class="margin_term"><a class="glossterm">active site</a><span class="glossdef">The location on an enzyme where a substrate binds and is transformed to product.</span></span> of the enzyme (<a class="xref" href="#gob-ch18_s06_f01">Figure 18.10 "Substrate Binding to the Active Site of an Enzyme"</a>). It possesses a unique conformation (including correctly positioned bonding groups) that is complementary to the structure of the substrate, so that the enzyme and substrate molecules fit together in much the same manner as a key fits into a tumbler lock. In fact, an early model describing the formation of the enzyme-substrate complex was called the <span class="margin_term"><a class="glossterm">lock-and-key model</a><span class="glossdef">A model that portrays an enzyme as conformationally rigid and able to bond only to a substrate or substrates that exactly fit the active site.</span></span> (<a class="xref" href="#gob-ch18_s06_f02">Figure 18.11 "The Lock-and-Key Model of Enzyme Action"</a>). This model portrayed the enzyme as conformationally rigid and able to bond only to substrates that exactly fit the active site.</p>
    <div class="figure large editable block" id="gob-ch18_s06_f01">
        <p class="title"><span class="title-prefix">Figure 18.10</span> Substrate Binding to the Active Site of an Enzyme</p>
        <img src="section_21/11aacd29092c2bf6edd10e57800cb799.jpg">
        <p class="para">The enzyme dihydrofolate reductase is shown with one of its substrates: NADP<sup class="superscript">+</sup> (a) unbound and (b) bound. The NADP<sup class="superscript">+</sup> (shown in red) binds to a pocket that is complementary to it in shape and ionic properties.</p>
    </div>
    <div class="figure large editable block" id="gob-ch18_s06_f02">
        <p class="title"><span class="title-prefix">Figure 18.11</span> The Lock-and-Key Model of Enzyme Action</p>
        <img src="section_21/494a8e8421b688f627e00d62c7fd6522.jpg">
        <p class="para">(a) Because the substrate and the active site of the enzyme have complementary structures and bonding groups, they fit together as a key fits a lock. (b) The catalytic reaction occurs while the two are bonded together in the enzyme-substrate complex.</p>
    </div>
    <p class="para editable block" id="gob-ch18_s06_p03">Working out the precise three-dimensional structures of numerous enzymes has enabled chemists to refine the original lock-and-key model of enzyme actions. They discovered that the binding of a substrate often leads to a large conformational change in the enzyme, as well as to changes in the structure of the substrate or substrates. The current theory, known as the <span class="margin_term"><a class="glossterm">induced-fit model</a><span class="glossdef">A model that says an enzyme can undergo a conformational change when it binds substrate molecules.</span></span>, says that enzymes can undergo a change in conformation when they bind substrate molecules, and the active site has a shape complementary to that of the substrate only <em class="emphasis">after</em> the substrate is bound, as shown for hexokinase in <a class="xref" href="#gob-ch18_s06_f03">Figure 18.12 "The Induced-Fit Model of Enzyme Action"</a>. After catalysis, the enzyme resumes its original structure.</p>
    <div class="figure large editable block" id="gob-ch18_s06_f03">
        <p class="title"><span class="title-prefix">Figure 18.12</span> The Induced-Fit Model of Enzyme Action</p>
        <img src="section_21/a3d4da672e302a0ef8ad2105d595a340.jpg">
        <p class="para">(a) The enzyme hexokinase without its substrate (glucose, shown in red) is bound to the active site. (b) The enzyme conformation changes dramatically when the substrate binds to it, resulting in additional interactions between hexokinase and glucose.</p>
    </div>
    <p class="para editable block" id="gob-ch18_s06_p04">The structural changes that occur when an enzyme and a substrate join together bring specific parts of a substrate into alignment with specific parts of the enzyme’s active site. Amino acid side chains in or near the binding site can then act as acid or base catalysts, provide binding sites for the transfer of functional groups from one substrate to another or aid in the rearrangement of a substrate. The participating amino acids, which are usually widely separated in the primary sequence of the protein, are brought close together in the active site as a result of the folding and bending of the polypeptide chain or chains when the protein acquires its tertiary and quaternary structure. Binding to enzymes brings reactants close to each other and aligns them properly, which has the same effect as increasing the concentration of the reacting compounds.</p>
    <div class="exercises editable block" id="gob-ch18_s06_n02">
        <h3 class="title">Example 1</h3>
        <ol class="orderedlist" id="gob-ch18_s06_l02">
            <li>What type of interaction would occur between an OH group present on a substrate molecule and a functional group in the active site of an enzyme?</li>
            <li>Suggest an amino acid whose side chain might be in the active site of an enzyme and form the type of interaction you just identified.</li>
        </ol>
        <p class="simpara">Solution</p>
        <ol class="orderedlist" id="gob-ch18_s06_l03">
            <li>An OH group would most likely engage in hydrogen bonding with an appropriate functional group present in the active site of an enzyme.</li>
            <li>Several amino acid side chains would be able to engage in hydrogen bonding with an OH group. One example would be asparagine, which has an amide functional group.</li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s06_qs01" condition="1col" defaultlabel="number">
        <h3 class="title">Skill-Building Exercise</h3>
        <ol class="qandadiv" id="gob-ch18_s06_qs01_qd01">
            <li class="qandaentry" id="gob-ch18_s06_qs01_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s06_qs01_p01">What type of interaction would occur between an COO<sup class="superscript">−</sup> group present on a substrate molecule and a functional group in the active site of an enzyme?</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s06_qs01_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s06_qs01_p03">Suggest an amino acid whose side chain might be in the active site of an enzyme and form the type of interaction you just identified.</p>
                </div>
            </li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s06_p07">One characteristic that distinguishes an enzyme from all other types of catalysts is its <em class="emphasis">substrate specificity</em>. An inorganic acid such as sulfuric acid can be used to increase the reaction rates of many different reactions, such as the hydrolysis of disaccharides, polysaccharides, lipids, and proteins, with complete impartiality. In contrast, enzymes are much more specific. Some enzymes act on a single substrate, while other enzymes act on any of a group of related molecules containing a similar functional group or chemical bond. Some enzymes even distinguish between D- and L-stereoisomers, binding one stereoisomer but not the other. Urease, for example, is an enzyme that catalyzes the hydrolysis of a single substrate—urea—but not the closely related compounds methyl urea, thiourea, or biuret. The enzyme carboxypeptidase, on the other hand, is far less specific. It catalyzes the removal of nearly any amino acid from the carboxyl end of any peptide or protein.</p>
    <div class="informalfigure large block">
        <img src="section_21/fde72914c160e46248abb55348300ca2.jpg">
    </div>
    <p class="para editable block" id="gob-ch18_s06_p08">Enzyme specificity results from the uniqueness of the active site in each different enzyme because of the identity, charge, and spatial orientation of the functional groups located there. It regulates cell chemistry so that the proper reactions occur in the proper place at the proper time. Clearly, it is crucial to the proper functioning of the living cell.</p>
    <div class="qandaset block" id="gob-ch18_s06_qs02" condition="1col" defaultlabel="number">
        <h3 class="title">Concept Review Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s06_qs02_qd01">
            <li class="qandaentry" id="gob-ch18_s06_qs02_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s06_qs02_p01">Distinguish between the lock-and-key model and induced-fit model of enzyme action.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s06_qs02_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s06_qs02_p03">Which enzyme has greater specificity—urease or carboxypeptidase? Explain.</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s06_qs02_ans" condition="1col" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s06_qs02_qd01_qa01_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s06_qs02_p02_ans">The lock-and-key model portrays an enzyme as conformationally rigid and able to bond only to substrates that exactly fit the active site. The induced fit model portrays the enzyme structure as more flexible and is complementary to the substrate only after the substrate is bound.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s06_qs02_qd01_qa02_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s06_qs02_p04_ans">Urease has the greater specificity because it can bind only to a single substrate. Carboxypeptidase, on the other hand, can catalyze the removal of nearly any amino acid from the carboxyl end of a peptide or protein.</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="key_takeaways editable block" id="gob-ch18_s06_n05">
        <h3 class="title">Key Takeaways</h3>
        <ul class="itemizedlist" id="gob-ch18_s06_l06">
            <li>A substrate binds to a specific region on an enzyme known as the active site, where the substrate can be converted to product.</li>
            <li>The substrate binds to the enzyme primarily through hydrogen bonding and other electrostatic interactions.</li>
            <li>The induced-fit model says that an enzyme can undergo a conformational change when binding a substrate.</li>
            <li>Enzymes exhibit varying degrees of substrate specificity.</li>
        </ul>
    </div>
    <div class="qandaset block" id="gob-ch18_s06_qs03" defaultlabel="number">
        <h3 class="title">Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s06_qs03_qd01">
            <li class="qandaentry" id="gob-ch18_s06_qs03_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s06_qs03_p01">What type of interaction would occur between each group present on a substrate molecule and a functional group of the active site in an enzyme?</p>
                    <ol class="orderedlist" id="gob-ch18_s06_qs03_l01" numeration="loweralpha">
                        <li>COOH</li>
                        <li>NH<sub class="subscript">3</sub><sup class="superscript">+</sup>
</li>
                        <li>OH</li>
                        <li>CH(CH<sub class="subscript">3</sub>)<sub class="subscript">2</sub>
</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s06_qs03_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s06_qs03_p02">What type of interaction would occur between each group present on a substrate molecule and a functional group of the active site in an enzyme?</p>
                    <ol class="orderedlist" id="gob-ch18_s06_qs03_l03" numeration="loweralpha">
                        <li>SH</li>
                        <li>NH<sub class="subscript">2</sub>
</li>
                        <li>C<sub class="subscript">6</sub>H<sub class="subscript">5</sub>
</li>
                        <li>COO<sup class="superscript">−</sup>
</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s06_qs03_qd01_qa03">
                <div class="question">
                    <p class="para" id="gob-ch18_s06_qs03_p03">For each functional group in Exercise 1, suggest an amino acid whose side chain might be in the active site of an enzyme and form the type of interaction you identified.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s06_qs03_qd01_qa04">
                <div class="question">
                    <p class="para" id="gob-ch18_s06_qs03_p05">For each functional group in Exercise 2, suggest an amino acid whose side chain might be in the active site of an enzyme and form the type of interaction you identified.</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s06_qs03_ans" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s06_qs03_qd01_qa01_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s06_qs03_l02_ans" numeration="loweralpha">                            <li>hydrogen bonding</li>
                            <li>ionic bonding</li>
                            <li>hydrogen bonding</li>
                            <li>dispersion forces</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s06_qs03_qd01_qa02_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s06_qs03_qd01_qa03_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s06_qs03_l05_ans" numeration="loweralpha">                            <li>The amino acid has a polar side chain capable of engaging in hydrogen bonding; serine (answers will vary).</li>
                            <li>The amino acid has a negatively charged side chain; aspartic acid (answers will vary).</li>
                            <li>The amino acid has a polar side chain capable of engaging in hydrogen bonding; asparagine (answers will vary).</li>
                            <li>The amino acid has a nonpolar side chain; isoleucine (answers will vary).</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s06_qs03_qd01_qa04_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
        </ol>
    </div>
</div>
<div class="section" id="gob-ch18_s07" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.7</span> Enzyme Activity</h2>
    <div class="learning_objectives editable block" id="gob-ch18_s07_n01">
        <h3 class="title">Learning Objective</h3>
        <ol class="orderedlist" id="gob-ch18_s07_l01">
            <li>Describe how pH, temperature, and the concentration of an enzyme and its substrate influence enzyme activity.</li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s07_p01">The single most important property of enzymes is the ability to increase the rates of reactions occurring in living organisms, a property known as <em class="emphasis">catalytic activity</em>. Because most enzymes are proteins, their activity is affected by factors that disrupt protein structure, as well as by factors that affect catalysts in general. Factors that disrupt protein structure, as we saw in <a class="xref" href="gob-ch18_s04#gob-ch18_s04">Section 18.4 "Proteins"</a>, include temperature and pH; factors that affect catalysts in general include reactant or substrate concentration and catalyst or enzyme concentration. The activity of an enzyme can be measured by monitoring either the rate at which a substrate disappears or the rate at which a product forms.</p>
    <div class="section" id="gob-ch18_s07_s01">
        <h2 class="title editable block">Concentration of Substrate</h2>
        <p class="para editable block" id="gob-ch18_s07_s01_p01">In the presence of a given amount of enzyme, the rate of an enzymatic reaction increases as the substrate concentration increases until a limiting rate is reached, after which further increase in the substrate concentration produces no significant change in the reaction rate (part (a) of <a class="xref" href="#gob-ch18_s07_s01_f01">Figure 18.13 "Concentration versus Reaction Rate"</a>). At this point, so much substrate is present that essentially all of the enzyme active sites have substrate bound to them. In other words, the enzyme molecules are saturated with substrate. The excess substrate molecules cannot react until the substrate already bound to the enzymes has reacted and been released (or been released without reacting).</p>
        <div class="figure large editable block" id="gob-ch18_s07_s01_f01">
            <p class="title"><span class="title-prefix">Figure 18.13</span> Concentration versus Reaction Rate</p>
            <img src="section_21/fdd62bfd8ace9333c101b82f6dce23dc.jpg">
            <p class="para">(a) This graph shows the effect of substrate concentration on the rate of a reaction that is catalyzed by a fixed amount of enzyme. (b) This graph shows the effect of enzyme concentration on the reaction rate at a constant level of substrate.</p>
        </div>
        <p class="para editable block" id="gob-ch18_s07_s01_p02">Let’s consider an analogy. Ten taxis (enzyme molecules) are waiting at a taxi stand to take people (substrate) on a 10-minute trip to a concert hall, one passenger at a time. If only 5 people are present at the stand, the rate of their arrival at the concert hall is 5 people in 10 minutes. If the number of people at the stand is increased to 10, the rate increases to 10 arrivals in 10 minutes. With 20 people at the stand, the rate would still be 10 arrivals in 10 minutes. The taxis have been “saturated.” If the taxis could carry 2 or 3 passengers each, the same principle would apply. The rate would simply be higher (20 or 30 people in 10 minutes) before it leveled off.</p>
    </div>
    <div class="section" id="gob-ch18_s07_s02">
        <h2 class="title editable block">Concentration of Enzyme</h2>
        <p class="para editable block" id="gob-ch18_s07_s02_p01">When the concentration of the enzyme is significantly lower than the concentration of the substrate (as when the number of taxis is far lower than the number of waiting passengers), the rate of an enzyme-catalyzed reaction is directly dependent on the enzyme concentration (part (b) of <a class="xref" href="#gob-ch18_s07_s01_f01">Figure 18.13 "Concentration versus Reaction Rate"</a>). This is true for any catalyst; the reaction rate increases as the concentration of the catalyst is increased.</p>
    </div>
    <div class="section" id="gob-ch18_s07_s03">
        <h2 class="title editable block">Temperature</h2>
        <p class="para editable block" id="gob-ch18_s07_s03_p01">A general rule of thumb for most chemical reactions is that a temperature rise of 10°C approximately doubles the reaction rate. To some extent, this rule holds for all enzymatic reactions. After a certain point, however, an increase in temperature causes a decrease in the reaction rate, due to denaturation of the protein structure and disruption of the active site (part (a) of <a class="xref" href="#gob-ch18_s07_s03_f01">Figure 18.14 "Temperature and pH versus Concentration"</a>). For many proteins, denaturation occurs between 45°C and 55°C. Furthermore, even though an enzyme may appear to have a maximum reaction rate between 40°C and 50°C, most biochemical reactions are carried out at lower temperatures because enzymes are not stable at these higher temperatures and will denature after a few minutes.</p>
        <div class="figure large editable block" id="gob-ch18_s07_s03_f01">
            <p class="title"><span class="title-prefix">Figure 18.14</span> Temperature and pH versus Concentration</p>
            <img src="section_21/9de13b61d3f051cfd2e49c1abea2ab46.jpg">
            <p class="para">(a) This graph depicts the effect of temperature on the rate of a reaction that is catalyzed by a fixed amount of enzyme. (b) This graph depicts the effect of pH on the rate of a reaction that is catalyzed by a fixed amount of enzyme.</p>
        </div>
        <p class="para editable block" id="gob-ch18_s07_s03_p02">At 0°C and 100°C, the rate of enzyme-catalyzed reactions is nearly zero. This fact has several practical applications. We sterilize objects by placing them in boiling water, which denatures the enzymes of any bacteria that may be in or on them. We preserve our food by refrigerating or freezing it, which slows enzyme activity. When animals go into hibernation in winter, their body temperature drops, decreasing the rates of their metabolic processes to levels that can be maintained by the amount of energy stored in the fat reserves in the animals’ tissues.</p>
    </div>
    <div class="section" id="gob-ch18_s07_s04">
        <h2 class="title editable block">Hydrogen Ion Concentration (pH)</h2>
        <p class="para editable block" id="gob-ch18_s07_s04_p01">Because most enzymes are proteins, they are sensitive to changes in the hydrogen ion concentration or pH. Enzymes may be denatured by extreme levels of hydrogen ions (whether high or low); <em class="emphasis">any</em> change in pH, even a small one, alters the degree of ionization of an enzyme’s acidic and basic side groups and the substrate components as well. Ionizable side groups located in the active site must have a certain charge for the enzyme to bind its substrate. Neutralization of even one of these charges alters an enzyme’s catalytic activity.</p>
        <p class="para editable block" id="gob-ch18_s07_s04_p02">An enzyme exhibits maximum activity over the narrow pH range in which a molecule exists in its properly charged form. The median value of this pH range is called the <span class="margin_term"><a class="glossterm">optimum pH</a><span class="glossdef">The pH at which a particular enzyme exhibits maximum activity.</span></span> of the enzyme (part (b) of <a class="xref" href="#gob-ch18_s07_s03_f01">Figure 18.14 "Temperature and pH versus Concentration"</a>). With the notable exception of gastric juice (the fluids secreted in the stomach), most body fluids have pH values between 6 and 8. Not surprisingly, most enzymes exhibit optimal activity in this pH range. However, a few enzymes have optimum pH values outside this range. For example, the optimum pH for pepsin, an enzyme that is active in the stomach, is 2.0.</p>
        <div class="qandaset block" id="gob-ch18_s07_qs01" condition="1col" defaultlabel="number">
            <h3 class="title">Concept Review Exercises</h3>
            <ol class="qandadiv" id="gob-ch18_s07_qs01_qd01">
                <li class="qandaentry" id="gob-ch18_s07_qs01_qd01_qa01">
                    <div class="question">
                        <p class="para" id="gob-ch18_s07_qs01_p01">The concentration of substrate X is low. What happens to the rate of the enzyme-catalyzed reaction if the concentration of X is doubled?</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s07_qs01_qd01_qa02">
                    <div class="question">
                        <p class="para" id="gob-ch18_s07_qs01_p03">What effect does an increase in the enzyme concentration have on the rate of an enzyme-catalyzed reaction?</p>
                    </div>
                </li>
            </ol>
        </div>
        <div class="qandaset block" id="gob-ch18_s07_qs01_ans" condition="1col" defaultlabel="number">
            <h3 class="title">Answers</h3>
            <ol class="qandadiv">
                <li class="qandaentry" id="gob-ch18_s07_qs01_qd01_qa01_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s07_qs01_p02_ans">If the concentration of the substrate is low, increasing its concentration will increase the rate of the reaction.</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s07_qs01_qd01_qa02_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s07_qs01_p04_ans">An increase in the amount of enzyme will increase the rate of the reaction (provided sufficient substrate is present).</p>
                    </div>
                </li>
            </ol>
        </div>
        <div class="key_takeaways editable block" id="gob-ch18_s07_s04_n02">
            <h3 class="title">Key Takeaways</h3>
            <ul class="itemizedlist" id="gob-ch18_s07_s04_l02">
                <li>Initially, an increase in substrate concentration leads to an increase in the rate of an enzyme-catalyzed reaction. As the enzyme molecules become saturated with substrate, this increase in reaction rate levels off.</li>
                <li>The rate of an enzyme-catalyzed reaction increases with an increase in the concentration of an enzyme.</li>
                <li>At low temperatures, an increase in temperature increases the rate of an enzyme-catalyzed reaction. At higher temperatures, the protein is denatured, and the rate of the reaction dramatically decreases.</li>
                <li>An enzyme has an optimum pH range in which it exhibits maximum activity.</li>
            </ul>
        </div>
        <div class="qandaset block" id="gob-ch18_s07_qs02" defaultlabel="number">
            <h3 class="title">Exercises</h3>
            <ol class="qandadiv" id="gob-ch18_s07_qs02_qd01">
                <li class="qandaentry" id="gob-ch18_s07_qs02_qd01_qa01">
                    <div class="question">
                        <p class="para" id="gob-ch18_s07_qs02_p01">In non-enzyme-catalyzed reactions, the reaction rate increases as the concentration of reactant is increased. In an enzyme-catalyzed reaction, the reaction rate initially increases as the substrate concentration is increased but then begins to level off, so that the increase in reaction rate becomes less and less as the substrate concentration increases. Explain this difference.</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s07_qs02_qd01_qa02">
                    <div class="question">
                        <p class="para" id="gob-ch18_s07_qs02_p03">Why do enzymes become inactive at very high temperatures?</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s07_qs02_qd01_qa03">
                    <div class="question">
                        <p class="para" id="gob-ch18_s07_qs02_p05">An enzyme has an optimum pH of 7.4. What is most likely to happen to the activity of the enzyme if the pH drops to 6.3? Explain.</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s07_qs02_qd01_qa04">
                    <div class="question">
                        <p class="para" id="gob-ch18_s07_qs02_p07">An enzyme has an optimum pH of 7.2. What is most likely to happen to the activity of the enzyme if the pH increases to 8.5? Explain.</p>
                    </div>
                </li>
            </ol>
        </div>
        <div class="qandaset block" id="gob-ch18_s07_qs02_ans" defaultlabel="number">
            <h3 class="title">Answers</h3>
            <ol class="qandadiv">
                <li class="qandaentry" id="gob-ch18_s07_qs02_qd01_qa01_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s07_qs02_p02_ans">In an enzyme-catalyzed reaction, the substrate binds to the enzyme to form an enzyme-substrate complex. If more substrate is present than enzyme, all of the enzyme binding sites will have substrate bound, and further increases in substrate concentration cannot increase the rate.</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s07_qs02_qd01_qa02_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s07_qs02_qd01_qa03_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s07_qs02_p06_ans">The activity will decrease; a pH of 6.3 is more acidic than 7.4, and one or more key groups in the active site may bind a hydrogen ion, changing the charge on that group.</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s07_qs02_qd01_qa04_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
            </ol>
        </div>
    </div>
</div>
<div class="section" id="gob-ch18_s08" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.8</span> Enzyme Inhibition</h2>
    <div class="learning_objectives editable block" id="gob-ch18_s08_n01">
        <h3 class="title">Learning Objectives</h3>
        <ol class="orderedlist" id="gob-ch18_s08_l01">
            <li>Explain what an enzyme inhibitor is.</li>
            <li>Distinguish between reversible and irreversible inhibitors.</li>
            <li>Distinguish between competitive and noncompetitive inhibitors.</li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s08_p01">In <a class="xref" href="gob-ch18_s07#gob-ch18_s07">Section 18.7 "Enzyme Activity"</a>, we noted that enzymes are inactivated at high temperatures and by changes in pH. These are nonspecific factors that would inactivate any enzyme. The activity of enzymes can also be regulated by more specific inhibitors. Many compounds are poisons because they bind covalently to particular enzymes or kinds of enzymes and inactivate them (<a class="xref" href="#gob-ch18_s08_t01">Table 18.7 "Poisons as Enzyme Inhibitors"</a>).</p>
    <div class="table block" id="gob-ch18_s08_t01" frame="all">
        <p class="title"><span class="title-prefix">Table 18.7</span> Poisons as Enzyme Inhibitors</p>
        <table cellpadding="0" cellspacing="0">
            <thead>
                <tr>
                    <th>Poison</th>
                    <th>Formula</th>
                    <th>Example of Enzyme Inhibited</th>
                    <th>Action</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>arsenate</td>
                    <td>AsO<sub class="subscript">4</sub><sup class="superscript">3−</sup>
</td>
                    <td>glyceraldehyde 3-phosphate dehydrogenase</td>
                    <td>substitutes for phosphate</td>
                </tr>
                <tr>
                    <td>iodoacetate</td>
                    <td>ICH<sub class="subscript">2</sub>COO<sup class="superscript">−</sup>
</td>
                    <td>triose phosphate dehydrogenase</td>
                    <td>binds to cysteine SH group</td>
                </tr>
                <tr>
                    <td>diisopropylfluoro-phosphate (DIFP; a nerve poison)</td>
                    <td>
	            <div class="informalfigure medium">
                <img src="section_21/b546f6da32af9673ffab55380e23be99.jpg">
            </div>
</td>
                    <td>acetylcholinesterase</td>
                    <td>binds to serine OH group</td>
                </tr>
            </tbody>
        </table>
    </div>
    <div class="section" id="gob-ch18_s08_s01">
        <h2 class="title editable block">Irreversible Inhibition: Poisons</h2>
        <p class="para editable block" id="gob-ch18_s08_s01_p01">An <span class="margin_term"><a class="glossterm">irreversible inhibitor</a><span class="glossdef">A substance that inactivates an enzyme by bonding covalently to a specific group at the active site.</span></span> inactivates an enzyme by bonding covalently to a particular group at the active site. The inhibitor-enzyme bond is so strong that the inhibition cannot be reversed by the addition of excess substrate. The nerve gases, especially DIFP, irreversibly inhibit biological systems by forming an enzyme-inhibitor complex with a specific OH group of serine situated at the active sites of certain enzymes. The peptidases trypsin and chymotrypsin contain serine groups at the active site and are inhibited by DIFP.</p>
        <div class="informalfigure large block">
            <img src="section_21/5323c1d53012bce02bdc5417cd5c7dd9.jpg">
        </div>
    </div>
    <div class="section" id="gob-ch18_s08_s02">
        <h2 class="title editable block">Reversible Inhibition</h2>
        <p class="para editable block" id="gob-ch18_s08_s02_p01">A <span class="margin_term"><a class="glossterm">reversible inhibitor</a><span class="glossdef">A substance that inactivates an enzyme by binding at the active site through noncovalent, reversible interactions.</span></span> inactivates an enzyme through noncovalent, more easily reversed, interactions. Unlike an irreversible inhibitor, a reversible inhibitor can dissociate from the enzyme. Reversible inhibitors include competitive inhibitors and noncompetitive inhibitors. (There are additional types of reversible inhibitors.) A <span class="margin_term"><a class="glossterm">competitive inhibitor</a><span class="glossdef">A compound that resembles a particular substrate and competes with the substrate for binding at the active site of an enzyme to slow the rate of the reaction.</span></span> is any compound that bears a structural resemblance to a particular substrate and thus competes with that substrate for binding at the active site of an enzyme. The inhibitor is not acted on by the enzyme but does prevent the substrate from approaching the active site.</p>
        <p class="para editable block" id="gob-ch18_s08_s02_p02">The degree to which a competitive inhibitor interferes with an enzyme’s activity depends on the relative concentrations of the substrate and the inhibitor. If the inhibitor is present in relatively large quantities, it will initially block most of the active sites. But because the binding is reversible, some substrate molecules will eventually bind to the active site and be converted to product. Increasing the substrate concentration promotes displacement of the inhibitor from the active site. Competitive inhibition can be completely reversed by adding substrate so that it reaches a much higher concentration than that of the inhibitor.</p>
        <p class="para editable block" id="gob-ch18_s08_s02_p03">Studies of competitive inhibition have provided helpful information about certain enzyme-substrate complexes and the interactions of specific groups at the active sites. As a result, pharmaceutical companies have synthesized drugs that competitively inhibit metabolic processes in bacteria and certain cancer cells. Many drugs are competitive inhibitors of specific enzymes.</p>
        <p class="para editable block" id="gob-ch18_s08_s02_p04">A classic example of competitive inhibition is the effect of malonate on the enzyme activity of succinate dehydrogenase (<a class="xref" href="#gob-ch18_s08_s02_f01">Figure 18.15 "Competitive Inhibition"</a>). Malonate and succinate are the anions of dicarboxylic acids and contain three and four carbon atoms, respectively. The malonate molecule binds to the active site because the spacing of its carboxyl groups is not greatly different from that of succinate. However, no catalytic reaction occurs because malonate does not have a CH<sub class="subscript">2</sub>CH<sub class="subscript">2</sub> group to convert to CH=CH. This reaction will also be discussed in connection with the Krebs cycle and energy production. (For more information about the Krebs cycle, see <a class="xref" href="gob-ch20#gob-ch20">Chapter 20 "Energy Metabolism"</a>, <a class="xref" href="gob-ch20_s04#gob-ch20_s04">Section 20.4 "Stage III of Catabolism"</a>.)</p>
        <div class="figure large editable block" id="gob-ch18_s08_s02_f01">
            <p class="title"><span class="title-prefix">Figure 18.15</span> Competitive Inhibition</p>
            <img src="section_21/c4856898d875b151ce6627716c8a6dc3.jpg">
            <p class="para">(a) Succinate binds to the enzyme succinate dehydrogenase. A dehydrogenation reaction occurs, and the product—fumarate—is released from the enzyme. (b) Malonate also binds to the active site of succinate dehydrogenase. In this case, however, no subsequent reaction occurs while malonate remains bound to the enzyme.</p>
        </div>
        <div class="callout block" id="gob-ch18_s08_s02_n01">
            <h3 class="title">To Your Health: Penicillin</h3>
            <p class="para" id="gob-ch18_s08_s02_p05"><em class="emphasis">Chemotherapy</em> is the strategic use of chemicals (that is, drugs) to destroy infectious microorganisms or cancer cells without causing excessive damage to the other, healthy cells of the host. From bacteria to humans, the metabolic pathways of all living organisms are quite similar, so the search for safe and effective chemotherapeutic agents is a formidable task. Many well-established chemotherapeutic drugs function by inhibiting a critical enzyme in the cells of the invading organism.</p>
            <p class="para" id="gob-ch18_s08_s02_p06">An <em class="emphasis">antibiotic</em> is a compound that kills bacteria; it may come from a natural source such as molds or be synthesized with a structure analogous to a naturally occurring antibacterial compound. Antibiotics constitute no well-defined class of chemically related substances, but many of them work by effectively inhibiting a variety of enzymes essential to bacterial growth.</p>
            <p class="para" id="gob-ch18_s08_s02_p07">Penicillin, one of the most widely used antibiotics in the world, was fortuitously discovered by Alexander Fleming in 1928, when he noticed antibacterial properties in a mold growing on a bacterial culture plate. In 1938, Ernst Chain and Howard Florey began an intensive effort to isolate penicillin from the mold and study its properties. The large quantities of penicillin needed for this research became available through development of a corn-based nutrient medium that the mold loved and through the discovery of a higher-yielding strain of mold at a United States Department of Agriculture research center near Peoria, Illinois. Even so, it was not until 1944 that large quantities of penicillin were being produced and made available for the treatment of bacterial infections.</p>
            <p class="para" id="gob-ch18_s08_s02_p08">Penicillin functions by interfering with the synthesis of cell walls of reproducing bacteria. It does so by inhibiting an enzyme—transpeptidase—that catalyzes the last step in bacterial cell-wall biosynthesis. The defective walls cause bacterial cells to burst. Human cells are not affected because they have cell membranes, not cell walls.</p>
            <p class="para" id="gob-ch18_s08_s02_p09">Several naturally occurring penicillins have been isolated. They are distinguished by different R groups connected to a common structure: a four-member cyclic amide (called a lactam ring) fused to a five-member ring. The addition of appropriate organic compounds to the culture medium leads to the production of the different kinds of penicillin.</p>
            <p class="para" id="gob-ch18_s08_s02_p10">The penicillins are effective against gram-positive bacteria (bacteria capable of being stained by Gram’s stain) and a few gram-negative bacteria (including the intestinal bacterium <em class="emphasis">Escherichia coli</em>). They are effective in the treatment of diphtheria, gonorrhea, pneumonia, syphilis, many pus infections, and certain types of boils. Penicillin G was the earliest penicillin to be used on a wide scale. However, it cannot be administered orally because it is quite unstable; the acidic pH of the stomach converts it to an inactive derivative. The major oral penicillins—penicillin V, ampicillin, and amoxicillin—on the other hand, are acid stable.</p>
            <div class="informalfigure large">
                <img src="section_21/007428424c06bf2f67ee3d2115a156b8.jpg">
            </div>
            <p class="para" id="gob-ch18_s08_s02_p11">Some strains of bacteria become resistant to penicillin through a mutation that allows them to synthesize an enzyme—penicillinase—that breaks the antibiotic down (by cleavage of the amide linkage in the lactam ring). To combat these strains, scientists have synthesized penicillin analogs (such as methicillin) that are not inactivated by penicillinase.</p>
            <p class="para" id="gob-ch18_s08_s02_p12">Some people (perhaps 5% of the population) are allergic to penicillin and therefore must be treated with other antibiotics. Their allergic reaction can be so severe that a fatal coma may occur if penicillin is inadvertently administered to them. Fortunately, several other antibiotics have been discovered. Most, including aureomycin and streptomycin, are the products of microbial synthesis. Others, such as the semisynthetic penicillins and tetracyclines, are made by chemical modifications of antibiotics; and some, like chloramphenicol, are manufactured entirely by chemical synthesis. They are as effective as penicillin in destroying infectious microorganisms. Many of these antibiotics exert their effects by blocking protein synthesis in microorganisms.</p>
            <p class="para" id="gob-ch18_s08_s02_p13">Initially, antibiotics were considered miracle drugs, substantially reducing the number of deaths from blood poisoning, pneumonia, and other infectious diseases. Some seven decades ago, a person with a major infection almost always died. Today, such deaths are rare. Seven decades ago, pneumonia was a dreaded killer of people of all ages. Today, it kills only the very old or those ill from other causes. Antibiotics have indeed worked miracles in our time, but even miracle drugs have limitations. Not long after the drugs were first used, disease organisms began to develop strains resistant to them. In a race to stay ahead of resistant bacterial strains, scientists continue to seek new antibiotics. The penicillins have now been partially displaced by related compounds, such as the cephalosporins and vancomycin. Unfortunately, some strains of bacteria have already shown resistance to these antibiotics.</p>
        </div>
        <p class="para editable block" id="gob-ch18_s08_s02_p14">Some reversible inhibitors are noncompetitive. A <span class="margin_term"><a class="glossterm">noncompetitive inhibitor</a><span class="glossdef">A compound that can combine with either the free enzyme or the enzyme-substrate complex at a site distinct from the active site to slow the rate of the reaction.</span></span> can combine with either the free enzyme or the enzyme-substrate complex because its binding site on the enzyme is distinct from the active site. Binding of this kind of inhibitor alters the three-dimensional conformation of the enzyme, changing the configuration of the active site with one of two results. Either the enzyme-substrate complex does not form at its normal rate, or, once formed, it does not yield products at the normal rate. Because the inhibitor does not structurally resemble the substrate, the addition of excess substrate does <em class="emphasis">not</em> reverse the inhibitory effect.</p>
        <p class="para editable block" id="gob-ch18_s08_s02_p15"><span class="margin_term"><a class="glossterm">Feedback inhibition</a><span class="glossdef">A normal biochemical process that makes use of noncompetitive inhibitors to control some enzymatic activity.</span></span> is a normal biochemical process that makes use of noncompetitive inhibitors to control some enzymatic activity. In this process, the final product inhibits the enzyme that catalyzes the first step in a series of reactions. Feedback inhibition is used to regulate the synthesis of many amino acids. For example, bacteria synthesize isoleucine from threonine in a series of five enzyme-catalyzed steps. As the concentration of isoleucine increases, some of it binds as a noncompetitive inhibitor to the first enzyme of the series (threonine deaminase), thus bringing about a decrease in the amount of isoleucine being formed (<a class="xref" href="#gob-ch18_s08_s02_f02">Figure 18.16 "Feedback Inhibition of Threonine Deaminase by Isoleucine"</a>).</p>
        <div class="figure large editable block" id="gob-ch18_s08_s02_f02">
            <p class="title"><span class="title-prefix">Figure 18.16</span> Feedback Inhibition of Threonine Deaminase by Isoleucine</p>
            <img src="section_21/9734a011dd765df8c8c6de30c7af2eee.jpg">
            <p class="para">Threonine deaminase is the first enzyme in the conversion of threonine to isoleucine. Isoleucine inhibits threonine deaminase through feedback inhibition .</p>
        </div>
        <div class="qandaset block" id="gob-ch18_s08_qs01" condition="1col" defaultlabel="number">
            <h3 class="title">Concept Review Exercises</h3>
            <ol class="qandadiv" id="gob-ch18_s08_qs01_qd01">
                <li class="qandaentry" id="gob-ch18_s08_qs01_qd01_qa01">
                    <div class="question">
                        <p class="para" id="gob-ch18_s08_qs01_p01">What are the characteristics of an irreversible inhibitor?</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s08_qs01_qd01_qa02">
                    <div class="question">
                        <p class="para" id="gob-ch18_s08_qs01_p03">In what ways does a competitive inhibitor differ from a noncompetitive inhibitor?</p>
                    </div>
                </li>
            </ol>
        </div>
        <div class="qandaset block" id="gob-ch18_s08_qs01_ans" condition="1col" defaultlabel="number">
            <h3 class="title">Answers</h3>
            <ol class="qandadiv">
                <li class="qandaentry" id="gob-ch18_s08_qs01_qd01_qa01_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s08_qs01_p02_ans">It inactivates an enzyme by bonding covalently to a particular group at the active site.</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s08_qs01_qd01_qa02_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s08_qs01_p04_ans">A competitive inhibitor structurally resembles the substrate for a given enzyme and competes with the substrate for binding at the active site of the enzyme. A noncompetitive inhibitor binds at a site distinct from the active site and can bind to either the free enzyme or the enzyme-substrate complex.</p>
                    </div>
                </li>
            </ol>
        </div>
        <div class="key_takeaways editable block" id="gob-ch18_s08_s02_n03">
            <h3 class="title">Key Takeaways</h3>
            <ul class="itemizedlist" id="gob-ch18_s08_s02_l02">
                <li>An irreversible inhibitor inactivates an enzyme by bonding covalently to a particular group at the active site.</li>
                <li>A reversible inhibitor inactivates an enzyme through noncovalent, reversible interactions.</li>
                <li>A competitive inhibitor competes with the substrate for binding at the active site of the enzyme.</li>
                <li>A noncompetitive inhibitor binds at a site distinct from the active site.</li>
            </ul>
        </div>
        <div class="qandaset block" id="gob-ch18_s08_qs02" condition="1col" defaultlabel="number">
            <h3 class="title">Exercises</h3>
            <ol class="qandadiv" id="gob-ch18_s08_qs02_qd01">
                <li class="qandaentry" id="gob-ch18_s08_qs02_qd01_qa01">
                    <div class="question">
                        <p class="para" id="gob-ch18_s08_qs02_p01">What amino acid is present in the active site of all enzymes that are irreversibly inhibited by nerve gases such as DIFP?</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s08_qs02_qd01_qa02">
                    <div class="question">
                        <p class="para" id="gob-ch18_s08_qs02_p03">Oxaloacetate (OOCCH<sub class="subscript">2</sub>COCOO) inhibits succinate dehydrogenase. Would you expect oxaloacetate to be a competitive or noncompetitive inhibitor? Explain.</p>
                    </div>
                </li>
            </ol>
        </div>
        <div class="qandaset block" id="gob-ch18_s08_qs02_ans" condition="1col" defaultlabel="number">
            <h3 class="title">Answer</h3>
            <ol class="qandadiv">
                <li class="qandaentry" id="gob-ch18_s08_qs02_qd01_qa01_ans">
                    <div class="answer">
                        <p class="para" id="gob-ch18_s08_qs02_p02_ans">serine</p>
                    </div>
                </li>
                <li class="qandaentry" id="gob-ch18_s08_qs02_qd01_qa02_ans" audience="instructoronly">
                    <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                    </div>
                </li>
            </ol>
        </div>
    </div>
</div>
<div class="section" id="gob-ch18_s09" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.9</span> Enzyme Cofactors and Vitamins</h2>
    <div class="learning_objectives editable block" id="gob-ch18_s09_n01">
        <h3 class="title">Learning Objective</h3>
        <ol class="orderedlist" id="gob-ch18_s09_l01">
            <li>Explain why vitamins are necessary in the diet.</li>
        </ol>
    </div>
    <p class="para editable block" id="gob-ch18_s09_p01">Many enzymes are simple proteins consisting entirely of one or more amino acid chains. Other enzymes contain a nonprotein component called a <span class="margin_term"><a class="glossterm">cofactor</a><span class="glossdef">A nonprotein component of an enzyme that is necessary for an enzyme’s proper functioning.</span></span> that is necessary for the enzyme’s proper functioning. There are two types of cofactors: inorganic ions [e.g., zinc or Cu(I) ions] and organic molecules known as <span class="margin_term"><a class="glossterm">coenzymes</a><span class="glossdef">A cofactor that is an organic molecule.</span></span>. Most coenzymes are vitamins or are derived from vitamins.</p>
    <p class="para editable block" id="gob-ch18_s09_p02"><span class="margin_term"><a class="glossterm">Vitamins</a><span class="glossdef">An organic compound that is essential in very small amounts for the maintenance of normal metabolism.</span></span> are organic compounds that are essential in very small (trace) amounts for the maintenance of normal metabolism. They generally cannot be synthesized at adequate levels by the body and must be obtained from the diet. The absence or shortage of a vitamin may result in a vitamin-deficiency disease. In the first half of the 20th century, a major focus of biochemistry was the identification, isolation, and characterization of vitamins.</p>
    <p class="para editable block" id="gob-ch18_s09_p03">Despite accumulating evidence that people needed more than just carbohydrates, fats, and proteins in their diets for normal growth and health, it was not until the early 1900s that research established the need for trace nutrients in the diet.</p>
    <p class="para editable block" id="gob-ch18_s09_p04">Because organisms differ in their synthetic abilities, a substance that is a vitamin for one species may not be so for another. Over the past 100 years, scientists have identified and isolated 13 vitamins required in the human diet and have divided them into two broad categories: the <em class="emphasis">fat-soluble vitamins</em>, which include vitamins A, D, E, and K, and the <em class="emphasis">water-soluble vitamins</em>, which are the B complex vitamins and vitamin C. All fat-soluble vitamins contain a high proportion of hydrocarbon structural components. There are one or two oxygen atoms present, but the compounds as a whole are nonpolar. In contrast, water-soluble vitamins contain large numbers of electronegative oxygen and nitrogen atoms, which can engage in hydrogen bonding with water. Most water-soluble vitamins act as coenzymes or are required for the synthesis of coenzymes. The fat-soluble vitamins are important for a variety of physiological functions. The key vitamins and their functions are found in <a class="xref" href="#gob-ch18_s09_t01">Table 18.8 "Fat-Soluble Vitamins and Physiological Functions"</a> and <a class="xref" href="#gob-ch18_s09_t02">Table 18.9 "Water-Soluble Vitamins and Physiological Functions"</a>.</p>
    <div class="table block" id="gob-ch18_s09_t01" frame="all">
        <p class="title"><span class="title-prefix">Table 18.8</span> Fat-Soluble Vitamins and Physiological Functions</p>
        <table cellpadding="0" cellspacing="0">
            <thead>
                <tr>
                    <th>Vitamin</th>
                    <th>Physiological Function</th>
                    <th>Effect of Deficiency</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>vitamin A (retinol)</td>
                    <td>formation of vision pigments; differentiation of epithelial cells</td>
                    <td>night blindness; continued deficiency leads to total blindness</td>
                </tr>
                <tr>
                    <td>vitamin D (cholecalciferol)</td>
                    <td>increases the body’s ability to absorb calcium and phosphorus</td>
                    <td>osteomalacia (softening of the bones); known as rickets in children</td>
                </tr>
                <tr>
                    <td>vitamin E (tocopherol)</td>
                    <td>fat-soluble antioxidant</td>
                    <td>damage to cell membranes</td>
                </tr>
                <tr>
                    <td>vitamin K (phylloquinone)</td>
                    <td>formation of prothrombin, a key enzyme in the blood-clotting process</td>
                    <td>increases the time required for blood to clot</td>
                </tr>
            </tbody>
        </table>
    </div>
    <div class="table block" id="gob-ch18_s09_t02" frame="all">
        <p class="title"><span class="title-prefix">Table 18.9</span> Water-Soluble Vitamins and Physiological Functions</p>
        <table cellpadding="0" cellspacing="0">
            <thead>
                <tr>
                    <th>Vitamin</th>
                    <th>Coenzyme</th>
                    <th>Coenzyme Function</th>
                    <th>Deficiency Disease</th>
                </tr>
            </thead>
            <tbody>
                <tr>
                    <td>vitamin B<sub class="subscript">1</sub> (thiamine)</td>
                    <td>thiamine pyrophosphate</td>
                    <td>decarboxylation reactions</td>
                    <td>beri-beri</td>
                </tr>
                <tr>
                    <td>vitamin B<sub class="subscript">2</sub> (riboflavin)</td>
                    <td>flavin mononucleotide or flavin adenine dinucleotide</td>
                    <td>oxidation-reduction reactions involving two hydrogen atoms</td>
                    <td>—</td>
                </tr>
                <tr>
                    <td>vitamin B<sub class="subscript">3</sub> (niacin)</td>
                    <td>nicotinamide adenine dinucleotide or nicotinamide adenine dinucleotide phosphate</td>
                    <td>oxidation-reduction reactions involving the hydride ion (H<sup class="superscript">−</sup>)</td>
                    <td>pellagra</td>
                </tr>
                <tr>
                    <td>vitamin B<sub class="subscript">6</sub> (pyridoxine)</td>
                    <td>pyridoxal phosphate</td>
                    <td>variety of reactions including the transfer of amino groups</td>
                    <td>—</td>
                </tr>
                <tr>
                    <td>vitamin B<sub class="subscript">12</sub> (cyanocobalamin)</td>
                    <td>methylcobalamin or deoxyadenoxylcobalamin</td>
                    <td>intramolecular rearrangement reactions</td>
                    <td>pernicious anemia</td>
                </tr>
                <tr>
                    <td>biotin</td>
                    <td>biotin</td>
                    <td>carboxylation reactions</td>
                    <td>—</td>
                </tr>
                <tr>
                    <td>folic acid</td>
                    <td>tetrahydrofolate</td>
                    <td>carrier of one-carbon units such as the formyl group</td>
                    <td>anemia</td>
                </tr>
                <tr>
                    <td>pantothenic Acid</td>
                    <td>coenzyme A</td>
                    <td>carrier of acyl groups</td>
                    <td>—</td>
                </tr>
                <tr>
                    <td>vitamin C (ascorbic acid)</td>
                    <td>none</td>
                    <td>antioxidant; formation of collagen, a protein found in tendons, ligaments, and bone</td>
                    <td>scurvy</td>
                </tr>
            </tbody>
        </table>
    </div>
    <p class="para editable block" id="gob-ch18_s09_p05">Vitamins C and E, as well as the provitamin β-carotene can act as antioxidants in the body. <span class="margin_term"><a class="glossterm">Antioxidants</a><span class="glossdef">A substance that prevents oxidation.</span></span> prevent damage from free radicals, which are molecules that are highly reactive because they have unpaired electrons. Free radicals are formed not only through metabolic reactions involving oxygen but also by such environmental factors as radiation and pollution.</p>
    <div class="callout editable block" id="gob-ch18_s09_n02">
        <h3 class="title">Note</h3>
        <p class="para" id="gob-ch18_s09_p06">β-carotene is known as a provitamin because it can be converted to vitamin A in the body.</p>
    </div>
    <p class="para editable block" id="gob-ch18_s09_p07">Free radicals react most commonly react with lipoproteins and unsaturated fatty acids in cell membranes, removing an electron from those molecules and thus generating a new free radical. The process becomes a chain reaction that finally leads to the oxidative degradation of the affected compounds. Antioxidants react with free radicals to stop these chain reactions by forming a more stable molecule or, in the case of vitamin E, a free radical that is much less reactive. (Vitamin E is converted back to its original form through interaction with vitamin C.)</p>
    <div class="qandaset block" id="gob-ch18_s09_qs01" condition="1col" defaultlabel="number">
        <h3 class="title">Concept Review Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s09_qs01_qd01">
            <li class="qandaentry" id="gob-ch18_s09_qs01_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s09_qs01_p01">What is the difference between a cofactor and a coenzyme?</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs01_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s09_qs01_p03">How are vitamins related to coenzymes?</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s09_qs01_ans" condition="1col" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s09_qs01_qd01_qa01_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s09_qs01_p02_ans">A coenzyme is one type of cofactor. Coenzymes are organic molecules required by some enzymes for activity. A cofactor can be either a coenzyme or an inorganic ion.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs01_qd01_qa02_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s09_qs01_p04_ans">Coenzymes are synthesized from vitamins.</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="key_takeaways editable block" id="gob-ch18_s09_n04">
        <h3 class="title">Key Takeaways</h3>
        <ul class="itemizedlist" id="gob-ch18_s09_l03">
            <li>Vitamins are organic compounds that are essential in very small amounts for the maintenance of normal metabolism.</li>
            <li>Vitamins are divided into two broad categories: fat-soluble vitamins and water-soluble vitamins.</li>
            <li>Most water-soluble vitamins are needed for the formation of coenzymes, which are organic molecules needed by some enzymes for catalytic activity.</li>
        </ul>
    </div>
    <div class="qandaset block" id="gob-ch18_s09_qs02" defaultlabel="number">
        <h3 class="title">Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s09_qs02_qd01">
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s09_qs02_p01">Identify each vitamin as water soluble or fat soluble.</p>
                    <ol class="orderedlist" id="gob-ch18_s09_qs02_l01" numeration="loweralpha">
                        <li>vitamin D</li>
                        <li>vitamin C</li>
                        <li>vitamin B<sub class="subscript">12</sub>
</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s09_qs02_p02">Identify each vitamin as water soluble or fat soluble.</p>
                    <ol class="orderedlist" id="gob-ch18_s09_qs02_l03" numeration="loweralpha">
                        <li>niacin</li>
                        <li>cholecalciferol</li>
                        <li>biotin</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa03">
                <div class="question">
                    <p class="para" id="gob-ch18_s09_qs02_p03">What vitamin is needed to form each coenzyme?</p>
                    <ol class="orderedlist" id="gob-ch18_s09_qs02_l05" numeration="loweralpha">
                        <li>pyridoxal phosphate</li>
                        <li>flavin adenine dinucleotide</li>
                        <li>coenzyme A</li>
                        <li>nicotinamide adenine dinucleotide</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa04">
                <div class="question">
                    <p class="para" id="gob-ch18_s09_qs02_p04">What coenzyme is formed from each vitamin?</p>
                    <ol class="orderedlist" id="gob-ch18_s09_qs02_l07" numeration="loweralpha">
                        <li>niacin</li>
                        <li>thiamine</li>
                        <li>cyanocobalamin</li>
                        <li>pantothenic acid</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa05">
                <div class="question">
                    <p class="para" id="gob-ch18_s09_qs02_p05">What is the function of each vitamin or coenzyme?</p>
                    <ol class="orderedlist" id="gob-ch18_s09_qs02_l09" numeration="loweralpha">
                        <li>flavin adenine dinucleotide</li>
                        <li>vitamin A</li>
                        <li>biotin</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa06">
                <div class="question">
                    <p class="para" id="gob-ch18_s09_qs02_p06">What is the function of each vitamin or coenzyme?</p>
                    <ol class="orderedlist" id="gob-ch18_s09_qs02_l11" numeration="loweralpha">
                        <li>vitamin K</li>
                        <li>pyridoxal phosphate</li>
                        <li>tetrahydrofolate</li>
                    </ol>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s09_qs02_ans" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa01_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s09_qs02_l02_ans" numeration="loweralpha">                            <li>fat soluble</li>
                            <li>water soluble</li>
                            <li>water soluble</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa02_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa03_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s09_qs02_l06_ans" numeration="loweralpha">                            <li>vitamin B<sub class="subscript">6</sub> or pyridoxine</li>
                            <li>vitamin B<sub class="subscript">2</sub> or riboflavin</li>
                            <li>pantothenic acid</li>
                            <li>vitamin B<sub class="subscript">3</sub> or niacin</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa04_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa05_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s09_qs02_l10_ans" numeration="loweralpha">                            <li>needed by enzymes that catalyze oxidation-reduction reactions in which two hydrogen atoms are transferred</li>
                            <li>needed for the formation of vision pigments</li>
                            <li>needed by enzymes that catalyze carboxylation reactions</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s09_qs02_qd01_qa06_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
        </ol>
    </div>
</div>
<div class="section end-of-chapter" id="gob-ch18_s10" condition="start-of-chunk" version="5.0" lang="en">
    <h2 class="title editable block">
<span class="title-prefix">18.10</span> End-of-Chapter Material</h2>
    <div class="callout block" id="gob-ch18_s10_n01">
        <h3 class="title">Chapter Summary</h3>
        <p class="para" id="gob-ch18_s10_p01"><em class="emphasis">To ensure that you understand the material in this chapter, you should review the meanings of the bold terms in the following summary and ask yourself how they relate to the topics in the chapter.</em></p>
        <p class="para" id="gob-ch18_s10_p02">A <strong class="emphasis bold">protein</strong> is a large biological polymer synthesized from <strong class="emphasis bold">amino acids</strong>, which are carboxylic acids containing an α-amino group. Proteins have a variety of important roles in living organisms, yet they are made from the same 20 L-amino acids. About half of these amino acids, the <strong class="emphasis bold">essential amino acids</strong>, cannot be synthesized by the human body and must be obtained from the diet. In the solid state and in neutral solutions, amino acids exist as <strong class="emphasis bold">zwitterions</strong>, species that are charged but electrically neutral. In this form, they behave much like inorganic salts. Each amino acid belongs to one of four classes depending on the characteristics of its R group or amino acid side chain: nonpolar, polar but neutral, positively charged, and negatively charged. Depending on the conditions, amino acids can act as either acids or bases, which means that proteins act as buffers. The pH at which an amino acid exists as the zwitterion is called the <strong class="emphasis bold">isoelectric point (pI)</strong>.</p>
        <p class="para" id="gob-ch18_s10_p03">The amino acids in a protein are linked together by <strong class="emphasis bold">peptide bonds</strong>. Protein chains containing 10 or fewer amino acids are usually referred to as <strong class="emphasis bold">peptides</strong>, with a prefix such as di- or tri- indicating the number of amino acids. Chains containing more than 50 amino acid units are referred to as <em class="emphasis">proteins</em> or <strong class="emphasis bold">polypeptides</strong>. Proteins are classified globular or fibrous, depending on their structure and resulting solubility in water. <strong class="emphasis bold">Globular proteins</strong> are nearly spherical and are soluble in water; <strong class="emphasis bold">fibrous proteins</strong> have elongated or fibrous structures and are not soluble in water.</p>
        <p class="para" id="gob-ch18_s10_p04">Protein molecules can have as many as four levels of structure. The <strong class="emphasis bold">primary structure</strong> is the sequence of amino acids in the chain. The <strong class="emphasis bold">secondary structure</strong> is the arrangement of adjacent atoms in the peptide chain; the most common arrangements are α-helices or β-pleated sheets. The <strong class="emphasis bold">tertiary structure</strong> is the overall three-dimensional shape of the molecule that results from the way the chain bends and folds in on itself. Proteins that consist of more than one chain have <strong class="emphasis bold">quaternary structure</strong>, which is the way the multiple chains are packed together.</p>
        <p class="para" id="gob-ch18_s10_p05">Four types of intramolecular and intermolecular forces contribute to secondary, tertiary, and quaternary structure: (1) <strong class="emphasis bold">hydrogen bonding</strong> between an oxygen or a nitrogen atom and a hydrogen atom bound to an oxygen atom or a nitrogen atom, either on the same chain or on a neighboring chain; (2) <strong class="emphasis bold">ionic bonding</strong> between one positively charged side chain and one negatively charged side chain; (3) <strong class="emphasis bold">disulfide linkages</strong> between cysteine units; and (4) <strong class="emphasis bold">dispersion forces</strong> between nonpolar side chains.</p>
        <p class="para" id="gob-ch18_s10_p06">Because of their complexity, protein molecules are delicate and easy to disrupt. A <em class="emphasis">denatured</em> protein is one whose conformation has been changed, in a process called <strong class="emphasis bold">denaturation</strong>, so that it can no longer do its physiological job. A variety of conditions, such as heat, ultraviolet radiation, the addition of organic compounds, or changes in pH can denature a protein.</p>
        <p class="para" id="gob-ch18_s10_p07">An <strong class="emphasis bold">enzyme</strong> is an organic catalyst produced by a living cell. Enzymes are such powerful catalysts that the reactions they promote occur rapidly at body temperature. Without the help of enzymes, these reactions would require high temperatures and long reaction times.</p>
        <p class="para" id="gob-ch18_s10_p08">The molecule or molecules on which an enzyme acts are called its <strong class="emphasis bold">substrates</strong>. An enzyme has an <strong class="emphasis bold">active site</strong> where its substrate or substrates bind to form an enzyme-substrate complex. The reaction occurs, and product is released:</p>
        <span class="informalequation"><span class="mathphrase">E + S → E–S → E + P</span></span>
        <p class="para" id="gob-ch18_s10_p09">The original <strong class="emphasis bold">lock-and-key model</strong> of enzyme and substrate binding pictured a rigid enzyme of unchanging configuration binding to the appropriate substrate. The newer <strong class="emphasis bold">induced-fit model</strong> describes the enzyme active site as changing its conformation after binding to the substrate.</p>
        <p class="para" id="gob-ch18_s10_p10">Most enzymes have maximal activity in a narrow pH range centered on an <strong class="emphasis bold">optimum pH</strong>. In this pH range, the enzyme is correctly folded, and catalytic groups in the active site have the correct charge (positive, negative, or neutral). For most enzymes, the optimum pH is between 6 and 8.</p>
        <p class="para" id="gob-ch18_s10_p11">Substances that interfere with enzyme function are called inhibitors. An <strong class="emphasis bold">irreversible inhibitor</strong> inactivates enzymes by forming covalent bonds to the enzyme, while a <strong class="emphasis bold">reversible inhibitor</strong> inactivates an enzyme by a weaker, noncovalent interaction that is easier to disrupt. A <strong class="emphasis bold">competitive inhibitor</strong> is a reversible inhibitor that is structurally similar to the substrate and binds to the active site. When the inhibitor is bound, the substrate is blocked from the active site and no reaction occurs. Because the binding of such an inhibitor is reversible, a high substrate concentration will overcome the inhibition because it increases the likelihood of the substrate binding. A <strong class="emphasis bold">noncompetitive inhibitor</strong> binds reversibly at a site distinct from the active site. Thus, it can bind to either the enzyme or the enzyme-substrate complex. The inhibitor changes the conformation of the active site so that the enzyme cannot function properly. Noncompetitive inhibitors are important in <strong class="emphasis bold">feedback inhibition</strong>, in which the amount of product produced by a series of reactions is carefully controlled. The final product in a series of reactions acts as a noncompetitive inhibitor of the initial enzyme.</p>
        <p class="para" id="gob-ch18_s10_p12">Simple enzymes consist entirely of one or more amino acid chains. Complex enzymes are composed of one or more amino acid chains joined to <strong class="emphasis bold">cofactors</strong>—inorganic ions or organic <strong class="emphasis bold">coenzymes</strong>. <strong class="emphasis bold">Vitamins</strong> are organic compounds that are essential in very small amounts for the maintenance of normal metabolism and generally cannot be synthesized at adequate levels by the body. Vitamins are divided into two broad categories: <em class="emphasis">fat-soluble</em> vitamins and <em class="emphasis">water-soluble</em> vitamins. Many of the water-soluble vitamins are used for the synthesis of coenzymes.</p>
    </div>
    <div class="qandaset block" id="gob-ch18_s10_qs01" condition="1col" defaultlabel="number">
        <h3 class="title">Additional Exercises</h3>
        <ol class="qandadiv" id="gob-ch18_s10_qs01_qd01">
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa01">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p01">Draw the structure of the amino acid γ-aminobutyric acid (GABA). Would you expect to find GABA in the amino acid sequence of a protein? Explain.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa02">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p03">Draw the structure of the amino acid homocysteine (R group = CH<sub class="subscript">2</sub>CH<sub class="subscript">2</sub>SH). Would you expect to find homocysteine in the amino acid sequence of a protein? Justify your answer.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa03">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p05">Write equations to show how leucine can act as a buffer (that is, how it can neutralize added acid or base).</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa04">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p06">Write equations to show how isoleucine can act as a buffer (that is, how it can neutralize added acid or base).</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa05">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p07">Glutathione (γ-glutamylcysteinylglycine) is a tripeptide found in all cells of higher animals. It contains glutamic acid joined in an unusual peptide linkage involving the carboxyl group of the R group (known as γ-carboxyl group), rather than the usual carboxyl group (the α-carboxyl group). Draw the structure of glutathione.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa06">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p08">Draw the structure of the pentapeptide whose sequence is arg-his-gly-leu-asp. Identify which of the amino acids have R groups that can donate or gain hydrogen ions.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa07">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p09">Bradykinin is a peptide hormone composed of nine amino acids that lowers blood pressure. Its primary structure is arg-pro-pro-gly-phe-ser-pro-phe-arg. Would you expect bradykinin to be positively charged, negatively charged, or neutral at a pH of 6.0? Justify your answer.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa08">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p11">One of the neurotransmitters involved in pain sensation is a peptide called substance P, which is composed of 11 amino acids and is released by nerve-cell terminals in response to pain. Its primary structure is arg-pro-lys-pro-gln-gln-phe-phe-gly-leu-met. Would you expect this peptide to be positively charged, negatively charged, or neutral at a pH of 6.0? Justify your answer.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa09">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p13">Carbohydrates are incorporated into <em class="emphasis">glycoproteins</em>. Would you expect the incorporation of sugar units to increase or decrease the solubility of a protein? Justify your answer.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa10">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p15">Some proteins have phosphate groups attached through an ester linkage to the OH groups of serine, threonine, or tyrosine residues to form <em class="emphasis">phosphoproteins</em>. Would you expect the incorporation of a phosphate group to increase or decrease the solubility of a protein? Justify your answer.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa11">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p17">Refer to <a class="xref" href="gob-ch18_s05#gob-ch18_s05_t01">Table 18.5 "Classes of Enzymes"</a> and determine how each enzyme would be classified.</p>
                    <ol class="orderedlist" id="gob-ch18_s10_qs01_l01" numeration="loweralpha">
                        <li>the enzyme that catalyzes the conversion of ethanol to acetaldehyde</li>
                        <li>the enzyme that catalyzes the breakdown of glucose 6-phosphate to glucose and inorganic phosphate ion (water is also a reactant in this reaction)</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa12">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p18">Refer to <a class="xref" href="gob-ch18_s05#gob-ch18_s05_t01">Table 18.5 "Classes of Enzymes"</a> and determine how each enzyme would be classified.</p>
                    <ol class="orderedlist" id="gob-ch18_s10_qs01_l03" numeration="loweralpha">
                        <li>
                            <p class="para">the enzyme that catalyzes the removal of a carboxyl group from pyruvate to form acetate</p>
                            <div class="informalfigure large" condition="equation-image">
                                <img src="section_21/6b0b24797cd918241fe64bebdce23020.jpg">
                            </div>
                        </li>
                        <li>
                            <p class="para">the enzyme that catalyzes the rearrangement of 3-phosphoglycerate to form 2-phosphoglycerate</p>
                            <div class="informalfigure large" condition="equation-image">
                                <img src="section_21/144efc51460976550d593ee891a73d24.jpg">
                            </div>
                        </li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa13">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p19">The enzyme lysozyme has an aspartic acid residue in the active site. In acidic solution, the enzyme is inactive, but activity increases as the pH rises to around 6. Explain why.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa14">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p21">The enzyme lysozyme has a glutamic acid residue in the active site. At neutral pH (6–7), the enzyme is active, but activity decreases as the pH rises. Explain why.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa15">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p23">The activity of a purified enzyme is measured at a substrate concentration of 1.0 μM and found to convert 49 μmol of substrate to product in 1 min. The activity is measured at 2.0 μM substrate and found to convert 98 μmol of substrate to product/minute.</p>
                    <ol class="orderedlist" id="gob-ch18_s10_qs01_l05" numeration="loweralpha">
                        <li>When the substrate concentration is 100 μM, how much substrate would you predict is converted to product in 1 min? What if the substrate concentration were increased to 1,000 μM (1.0 mM)?</li>
                        <li>The activities actually measured are 676 μmol product formed/minute at a substrate concentration of 100 μM and 698 μmol product formed/minute at 1,000 μM (1.0 mM) substrate. Is there any discrepancy between these values and those you predicted in Exercise 15a? Explain.</li>
                    </ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa16">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p24">A patient has a fever of 39°C. Would you expect the activity of enzymes in the body to increase or decrease relative to their activity at normal body temperature (37°C)?</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa17">
                <div class="question">
                    <p class="para" id="gob-ch18_s10_qs01_p26">Using your knowledge of factors that influence enzyme activity, describe what happens when milk is pasteurized.</p>
                </div>
            </li>
        </ol>
    </div>
    <div class="qandaset block" id="gob-ch18_s10_qs01_ans" condition="1col" defaultlabel="number">
        <h3 class="title">Answers</h3>
        <ol class="qandadiv">
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa01_ans">
                <div class="answer">
                    <div class="informalfigure large">
                        <img src="section_21/2ee003d119a1ce96c3b2b9987e48809a.jpg">
                    </div>
                    <p class="para" id="gob-ch18_s10_qs01_p02_ans">This amino acid would not be found in proteins because it is not an α-amino acid.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa02_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa03_ans">
                <div class="answer">
                    <div class="informalfigure large">
                        <img src="section_21/314e46bc804ee7c8726e79d3b01f4907.jpg">
                    </div>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa04_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa05_ans">
                <div class="answer">
                    <div class="informalfigure large">
                        <img src="section_21/d17b4665241d7410aecc8e906b9f59e8.jpg">
                    </div>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa06_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa07_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s10_qs01_p10_ans">Bradykinin would be positively charged; all of the amino acids, except for arginine, have R groups that do not become either positively or negatively charged. The two arginines are R groups that are positively charged at neutral pH, so the peptide would have an overall positive charge.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa08_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa09_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s10_qs01_p14_ans">Carbohydrates have many OH groups attached, which can engage in hydrogen bonding with water, which increases the solubility of the proteins.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa10_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa11_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s10_qs01_l02_ans" numeration="loweralpha">                            <li>oxidoreductase</li>
                            <li>hydrolase</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa12_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa13_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s10_qs01_p20_ans">The enzyme is active when the carboxyl group in the R group of aspartic acid does not have the hydrogen attached (forming COO<sup class="superscript">–</sup>); the hydrogen is removed when the pH of the solution is around pH 6 or higher.</p>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa14_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa15_ans">
                <div class="answer">
                    <p class="para"></p>
<ol class="orderedlist" id="gob-ch18_s10_qs01_l06_ans" numeration="loweralpha">                            <li>at 100 μM, you would predict that the rate would increase 100 times to 4,900 μmol of substrate to product in 1 min; at 1.0 mM, you would predict an increase to 49,000 μmol of substrate to product in 1 min.</li>
                            <li>There is a great discrepancy between the predicted rates and actual rates; this occurs because the enzyme becomes saturated with substrate, preventing a further increase in the rate of the reaction (the reaction is no longer linear with respect to substrate concentration because it is at very low concentrations).</li>
</ol>
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa16_ans" audience="instructoronly">
                <div class="answer" audience="instructoronly" d="" html="http://www.w3.org/1999/xhtml" mml="http://www.w3.org/1998/Math/MathML" xlink="http://www.w3.org/1999/xlink" xml="http://www.w3.org/XML/1998/namespace">
                </div>
            </li>
            <li class="qandaentry" id="gob-ch18_s10_qs01_qd01_qa17_ans">
                <div class="answer">
                    <p class="para" id="gob-ch18_s10_qs01_p27_ans">When milk is pasteurized, it is heated to high temperatures. These high temperatures denature the proteins in bacteria, so they cannot carry out needed functions to grow and multiply.</p>
                </div>
            </li>
        </ol>
    </div>
</div>

  </div>
  
  <div id=navbar-bottom class="navbar">
    <div class="navbar-part left">
      
        <a href="s20-lipids.html"><img src="shared/images/batch-left.png"></a> <a href="s20-lipids.html">Previous Chapter</a>
      
    </div>
    <div class="navbar-part middle">
      <a href="index.html"><img src="shared/images/batch-up.png"></a> <a href="index.html">Table of Contents</a>
    </div>
    <div class="navbar-part right">
      
        <a href="s22-nucleic-acids.html">Next Chapter</a> <a href="s22-nucleic-acids.html"><img src="shared/images/batch-right.png"></a>
      
    </div>
  </div>

  </div>
  <script type="text/javascript" src="shared/book.js"></script>
  
  
</body>
</html>
